
<!doctype html>
<html>
<head>
<meta charset='UTF-8'>
<link href="gene.ico" rel="SHORTCUT ICON">
    <title>Docs | 3.Amino Acids and Peptides</title>
<link href='https://fonts.loli.net/css?family=Open+Sans:400italic,700italic,700,400&subset=latin,latin-ext' rel='stylesheet' type='text/css' /><style type='text/css'>html {overflow-x: initial !important;}:root { --mermaid-theme: default; --mermaid-sequence-numbers: off; --mermaid-flowchart-curve: linear; --mermaid--gantt-left-padding: 75; --sequence-theme: simple; }


:root { --bg-color:#ffffff; --text-color:#333333; --select-text-bg-color:#B5D6FC; --select-text-font-color:auto; --monospace:"Lucida Console",Consolas,"Courier",monospace; --title-bar-height:20px; }
.mac-os-11 { --title-bar-height:28px; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; inset: 0px; font-size: 1rem; line-height: 1.42857; overflow-x: hidden; background: inherit; tab-size: 4; }
iframe { margin: auto; }
a.url { word-break: break-all; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { text-shadow: none; background: var(--select-text-bg-color); color: var(--select-text-font-color); }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; overflow-wrap: break-word; position: relative; white-space: normal; overflow-x: visible; padding-top: 36px; }
#write.first-line-indent p { text-indent: 2em; }
#write.first-line-indent li p, #write.first-line-indent p * { text-indent: 0px; }
#write.first-line-indent li { margin-left: 2em; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
.typora-export .footnote-line, .typora-export li, .typora-export p { white-space: pre-wrap; }
.typora-export .task-list-item input { pointer-events: none; }
@media screen and (max-width: 500px) {
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  #write { padding-left: 20px; padding-right: 20px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
#write li > figure:last-child { margin-bottom: 0.5rem; }
#write ol, #write ul { position: relative; }
img { max-width: 100%; vertical-align: middle; image-orientation: from-image; }
button, input, select, textarea { color: inherit; font: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
*, ::after, ::before { box-sizing: border-box; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p, #write pre { width: inherit; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p { position: relative; }
p { line-height: inherit; }
h1, h2, h3, h4, h5, h6 { break-after: avoid-page; break-inside: avoid; orphans: 4; }
p { orphans: 4; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
.md-math-block, .md-rawblock, h1, h2, h3, h4, h5, h6, p { margin-top: 1rem; margin-bottom: 1rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: 700; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-radius: 4px; cursor: pointer; }
sup.md-footnote a, sup.md-footnote a:hover { color: inherit; text-transform: inherit; text-decoration: inherit; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; }
figure { overflow-x: auto; margin: 1.2em 0px; max-width: calc(100% + 16px); padding: 0px; }
figure > table { margin: 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 32px; }
.CodeMirror-gutters { border-right: 0px; background-color: inherit; }
.CodeMirror-linenumber { user-select: none; }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
#write pre { white-space: pre-wrap; }
#write.fences-no-line-wrapping pre { white-space: pre; }
#write pre.ty-contain-cm { white-space: normal; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background: inherit; position: relative !important; }
.md-fences-adv-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
#write .md-fences.mock-cm { white-space: pre-wrap; }
.md-fences.md-fences-with-lineno { padding-left: 0px; }
#write.fences-no-line-wrapping .md-fences.mock-cm { white-space: pre; overflow-x: auto; }
.md-fences.mock-cm.md-fences-with-lineno { padding-left: 8px; }
.CodeMirror-line, twitterwidget { break-inside: avoid; }
.footnotes { opacity: 0.8; font-size: 0.9rem; margin-top: 1em; margin-bottom: 1em; }
.footnotes + .footnotes { margin-top: 0px; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; background: 0px 0px; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; -webkit-tap-highlight-color: transparent; line-height: normal; font-weight: 400; text-align: left; box-sizing: content-box; direction: ltr; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li .mathjax-block, li p { margin: 0.5rem 0px; }
li blockquote { margin: 1rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child, li > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; white-space: normal; }
#write .footnote-line { white-space: pre-wrap; }
@media print {
  body, html { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; font-variant-ligatures: no-common-ligatures; }
  #write { margin-top: 0px; padding-top: 0px; border-color: transparent !important; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  .typora-export #write { break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  .is-mac table { break-inside: avoid; }
  .typora-export-show-outline .typora-export-sidebar { display: none; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 0.8rem; white-space: pre-wrap; background: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p > .md-image:only-child:not(.md-img-error) img, p > img:only-child { display: block; margin: auto; }
#write.first-line-indent p > .md-image:only-child:not(.md-img-error) img { left: -2em; position: relative; }
p > .md-image:only-child { display: inline-block; width: 100%; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.md-math-block { width: 100%; }
.md-math-block:not(:empty)::after { display: none; }
.MathJax_ref { fill: currentcolor; }
[contenteditable="true"]:active, [contenteditable="true"]:focus, [contenteditable="false"]:active, [contenteditable="false"]:focus { outline: 0px; box-shadow: none; }
.md-task-list-item { position: relative; list-style-type: none; }
.task-list-item.md-task-list-item { padding-left: 0px; }
.md-task-list-item > input { position: absolute; top: 0px; left: 0px; margin-left: -1.2em; margin-top: calc(1em - 10px); border: none; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc-content::after, .md-toc::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { text-decoration: underline; }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: 700; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) {
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
code, pre, samp, tt { font-family: var(--monospace); }
kbd { margin: 0px 0.1em; padding: 0.1em 0.6em; font-size: 0.8em; color: rgb(36, 39, 41); background: rgb(255, 255, 255); border: 1px solid rgb(173, 179, 185); border-radius: 3px; box-shadow: rgba(12, 13, 14, 0.2) 0px 1px 0px, rgb(255, 255, 255) 0px 0px 0px 2px inset; white-space: nowrap; vertical-align: middle; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.6; font-family: var(--monospace); }
code { text-align: left; vertical-align: initial; }
a.md-print-anchor { white-space: pre !important; border-width: initial !important; border-style: none !important; border-color: initial !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: 0px !important; background: 0px 0px !important; text-decoration: initial !important; text-shadow: initial !important; }
.os-windows.monocolor-emoji .md-emoji { font-family: "Segoe UI Symbol", sans-serif; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="flow"] svg, [lang="mermaid"] svg { max-width: 100%; height: auto; }
[lang="mermaid"] .node text { font-size: 1rem; }
table tr th { border-bottom: 0px; }
video { max-width: 100%; display: block; margin: 0px auto; }
iframe { max-width: 100%; width: 100%; border: none; }
.highlight td, .highlight tr { border: 0px; }
mark { background: rgb(255, 255, 0); color: rgb(0, 0, 0); }
.md-html-inline .md-plain, .md-html-inline strong, mark .md-inline-math, mark strong { color: inherit; }
.md-expand mark .md-meta { opacity: 0.3 !important; }
mark .md-meta { color: rgb(0, 0, 0); }
@media print {
  .typora-export h1, .typora-export h2, .typora-export h3, .typora-export h4, .typora-export h5, .typora-export h6 { break-inside: avoid; }
}
.md-diagram-panel .messageText { stroke: none !important; }
.md-diagram-panel .start-state { fill: var(--node-fill); }
.md-diagram-panel .edgeLabel rect { opacity: 1 !important; }
.md-fences.md-fences-math { font-size: 1em; }
.md-fences-advanced:not(.md-focus) { padding: 0px; white-space: nowrap; border: 0px; }
.md-fences-advanced:not(.md-focus) { background: inherit; }
.typora-export-show-outline .typora-export-content { max-width: 1440px; margin: auto; display: flex; flex-direction: row; }
.typora-export-sidebar {background-color:#183055; width: 260px; font-size: 0.8rem; color:white; margin-top: 80px; margin-right: 18px; }
.typora-export-show-outline #write { --webkit-flex:2; flex: 2 1 0%; }
.typora-export-sidebar .outline-content {background-color:#183055; position: fixed; top: 0px; max-height: 100%; overflow: hidden auto; padding-bottom: 30px; padding-top: 50px; width: 260px; }
@media screen and (max-width: 1440px) {
  .typora-export-sidebar, .typora-export-sidebar .outline-content { width: 230px; }
}
@media screen and (max-width: 950px) {
  .typora-export-sidebar { display: none; }
}
.outline-content li, .outline-content ul { margin-left: 0px; margin-right: 0px; padding-left: 0px; padding-right: 0px; list-style: none; }
.outline-content ul { margin-top: 0px; margin-bottom: 0px; }
.outline-content strong { font-weight: 400; }
.outline-expander { width: 1rem; height: 1.42857rem; position: relative; display: table-cell; vertical-align: middle; cursor: pointer; padding-left: 4px; }
.outline-expander::before { content: ""; position: relative; font-family: Ionicons; display: inline-block; font-size: 8px; vertical-align: middle; }
.outline-item { padding-top: 3px; padding-bottom: 3px; cursor: pointer; }
.outline-expander:hover::before { content: ""; }
.outline-h1 > .outline-item { padding-left: 0px; }
.outline-h2 > .outline-item { padding-left: 1em; }
.outline-h3 > .outline-item { padding-left: 2em; }
.outline-h4 > .outline-item { padding-left: 3em; }
.outline-h5 > .outline-item { padding-left: 4em; }
.outline-h6 > .outline-item { padding-left: 5em; }
.outline-label { cursor: pointer; display: table-cell; vertical-align: middle; text-decoration: none; color: inherit; }
.outline-label:hover { text-decoration: underline; }
.outline-item:hover { border-color: rgb(245, 245, 245); background-color: var(--item-hover-bg-color); }
.outline-item:hover { margin-left: -28px; margin-right: -28px; border-left: 28px solid transparent; border-right: 28px solid transparent; }
.outline-item-single .outline-expander::before, .outline-item-single .outline-expander:hover::before { display: none; }
.outline-item-open > .outline-item > .outline-expander::before { content: ""; }
.outline-children { display: none; }
.info-panel-tab-wrapper { display: none; }
.outline-item-open > .outline-children { display: block; }
.typora-export .outline-item { padding-top: 1px; padding-bottom: 1px; }
.typora-export .outline-item:hover { margin-right: -8px; border-right: 8px solid transparent; }
.typora-export .outline-expander::before { content: "+"; font-family: inherit; top: -1px; }
.typora-export .outline-expander:hover::before, .typora-export .outline-item-open > .outline-item > .outline-expander::before { content: "−"; }
.typora-export-collapse-outline .outline-children { display: none; }
.typora-export-collapse-outline .outline-item-open > .outline-children, .typora-export-no-collapse-outline .outline-children { display: block; }
.typora-export-no-collapse-outline .outline-expander::before { content: "" !important; }
.typora-export-show-outline .outline-item-active > .outline-item .outline-label { font-weight: 700; }
.md-inline-math-container mjx-container { zoom: 0.95; }


.CodeMirror { height: auto; }
.CodeMirror.cm-s-inner { background: inherit; }
.CodeMirror-scroll { overflow: auto hidden; z-index: 3; }
.CodeMirror-gutter-filler, .CodeMirror-scrollbar-filler { background-color: rgb(255, 255, 255); }
.CodeMirror-gutters { border-right: 1px solid rgb(221, 221, 221); background: inherit; white-space: nowrap; }
.CodeMirror-linenumber { padding: 0px 3px 0px 5px; text-align: right; color: rgb(153, 153, 153); }
.cm-s-inner .cm-keyword { color: rgb(119, 0, 136); }
.cm-s-inner .cm-atom, .cm-s-inner.cm-atom { color: rgb(34, 17, 153); }
.cm-s-inner .cm-number { color: rgb(17, 102, 68); }
.cm-s-inner .cm-def { color: rgb(0, 0, 255); }
.cm-s-inner .cm-variable { color: rgb(0, 0, 0); }
.cm-s-inner .cm-variable-2 { color: rgb(0, 85, 170); }
.cm-s-inner .cm-variable-3 { color: rgb(0, 136, 85); }
.cm-s-inner .cm-string { color: rgb(170, 17, 17); }
.cm-s-inner .cm-property { color: rgb(0, 0, 0); }
.cm-s-inner .cm-operator { color: rgb(152, 26, 26); }
.cm-s-inner .cm-comment, .cm-s-inner.cm-comment { color: rgb(170, 85, 0); }
.cm-s-inner .cm-string-2 { color: rgb(255, 85, 0); }
.cm-s-inner .cm-meta { color: rgb(85, 85, 85); }
.cm-s-inner .cm-qualifier { color: rgb(85, 85, 85); }
.cm-s-inner .cm-builtin { color: rgb(51, 0, 170); }
.cm-s-inner .cm-bracket { color: rgb(153, 153, 119); }
.cm-s-inner .cm-tag { color: rgb(17, 119, 0); }
.cm-s-inner .cm-attribute { color: rgb(0, 0, 204); }
.cm-s-inner .cm-header, .cm-s-inner.cm-header { color: rgb(0, 0, 255); }
.cm-s-inner .cm-quote, .cm-s-inner.cm-quote { color: rgb(0, 153, 0); }
.cm-s-inner .cm-hr, .cm-s-inner.cm-hr { color: rgb(153, 153, 153); }
.cm-s-inner .cm-link, .cm-s-inner.cm-link { color: rgb(0, 0, 204); }
.cm-negative { color: rgb(221, 68, 68); }
.cm-positive { color: rgb(34, 153, 34); }
.cm-header, .cm-strong { font-weight: 700; }
.cm-del { text-decoration: line-through; }
.cm-em { font-style: italic; }
.cm-link { text-decoration: underline; }
.cm-error { color: red; }
.cm-invalidchar { color: red; }
.cm-constant { color: rgb(38, 139, 210); }
.cm-defined { color: rgb(181, 137, 0); }
div.CodeMirror span.CodeMirror-matchingbracket { color: rgb(0, 255, 0); }
div.CodeMirror span.CodeMirror-nonmatchingbracket { color: rgb(255, 34, 34); }
.cm-s-inner .CodeMirror-activeline-background { background: inherit; }
.CodeMirror { position: relative; overflow: hidden; }
.CodeMirror-scroll { height: 100%; outline: 0px; position: relative; box-sizing: content-box; background: inherit; }
.CodeMirror-sizer { position: relative; }
.CodeMirror-gutter-filler, .CodeMirror-hscrollbar, .CodeMirror-scrollbar-filler, .CodeMirror-vscrollbar { position: absolute; z-index: 6; display: none; outline: 0px; }
.CodeMirror-vscrollbar { right: 0px; top: 0px; overflow: hidden; }
.CodeMirror-hscrollbar { bottom: 0px; left: 0px; overflow: auto hidden; }
.CodeMirror-scrollbar-filler { right: 0px; bottom: 0px; }
.CodeMirror-gutter-filler { left: 0px; bottom: 0px; }
.CodeMirror-gutters { position: absolute; left: 0px; top: 0px; padding-bottom: 10px; z-index: 3; overflow-y: hidden; }
.CodeMirror-gutter { white-space: normal; height: 100%; box-sizing: content-box; padding-bottom: 30px; margin-bottom: -32px; display: inline-block; }
.CodeMirror-gutter-wrapper { position: absolute; z-index: 4; background: 0px 0px !important; border: none !important; }
.CodeMirror-gutter-background { position: absolute; top: 0px; bottom: 0px; z-index: 4; }
.CodeMirror-gutter-elt { position: absolute; cursor: default; z-index: 4; }
.CodeMirror-lines { cursor: text; }
.CodeMirror pre { border-radius: 0px; border-width: 0px; background: 0px 0px; font-family: inherit; font-size: inherit; margin: 0px; white-space: pre; overflow-wrap: normal; color: inherit; z-index: 2; position: relative; overflow: visible; }
.CodeMirror-wrap pre { overflow-wrap: break-word; white-space: pre-wrap; word-break: normal; }
.CodeMirror-code pre { border-right: 30px solid transparent; width: fit-content; }
.CodeMirror-wrap .CodeMirror-code pre { border-right: none; width: auto; }
.CodeMirror-linebackground { position: absolute; inset: 0px; z-index: 0; }
.CodeMirror-linewidget { position: relative; z-index: 2; overflow: auto; }
.CodeMirror-wrap .CodeMirror-scroll { overflow-x: hidden; }
.CodeMirror-measure { position: absolute; width: 100%; height: 0px; overflow: hidden; visibility: hidden; }
.CodeMirror-measure pre { position: static; }
.CodeMirror div.CodeMirror-cursor { position: absolute; visibility: hidden; border-right: none; width: 0px; }
.CodeMirror div.CodeMirror-cursor { visibility: hidden; }
.CodeMirror-focused div.CodeMirror-cursor { visibility: inherit; }
.cm-searching { background: rgba(255, 255, 0, 0.4); }
span.cm-underlined { text-decoration: underline; }
span.cm-strikethrough { text-decoration: line-through; }
.cm-tw-syntaxerror { color: rgb(255, 255, 255); background-color: rgb(153, 0, 0); }
.cm-tw-deleted { text-decoration: line-through; }
.cm-tw-header5 { font-weight: 700; }
.cm-tw-listitem:first-child { padding-left: 10px; }
.cm-tw-box { border-style: solid; border-right-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-color: inherit; border-top-width: 0px !important; }
.cm-tw-underline { text-decoration: underline; }
@media print {
  .CodeMirror div.CodeMirror-cursor { visibility: hidden; }
}





:root {
    --side-bar-bg-color: #F5F5F5;
    --control-text-color: #777;
}

@include-when-export url(https://fonts.loli.net/css?family=Open+Sans:400italic,700italic,700,400&subset=latin,latin-ext);

/* open-sans-regular - latin-ext_latin */
  /* open-sans-italic - latin-ext_latin */
    /* open-sans-700 - latin-ext_latin */
    /* open-sans-700italic - latin-ext_latin */
  html {
    font-size: 16px;
    -webkit-font-smoothing: antialiased;
}

body {
    font-family: "Open Sans","Clear Sans", "Helvetica Neue", Helvetica, Arial, 'Segoe UI Emoji', sans-serif;
    color: rgb(51, 51, 51);
    line-height: 1.6;
}

#write {
    max-width: 860px;
  	margin: 0 auto;
  	padding: 30px;
    padding-bottom: 100px;
}

@media only screen and (min-width: 1400px) {
	#write {
		max-width: 1024px;
	}
}

@media only screen and (min-width: 1800px) {
	#write {
		max-width: 1200px;
	}
}

#write > ul:first-child,
#write > ol:first-child{
    margin-top: 30px;
}

a {
    color: #4183C4;
}
h1,
h2,
h3,
h4,
h5,
h6 {
    position: relative;
    margin-top: 1rem;
    margin-bottom: 1rem;
    font-weight: bold;
    line-height: 1.4;
    cursor: text;
}
h1:hover a.anchor,
h2:hover a.anchor,
h3:hover a.anchor,
h4:hover a.anchor,
h5:hover a.anchor,
h6:hover a.anchor {
    text-decoration: none;
}
h1 tt,
h1 code {
    font-size: inherit;
}
h2 tt,
h2 code {
    font-size: inherit;
}
h3 tt,
h3 code {
    font-size: inherit;
}
h4 tt,
h4 code {
    font-size: inherit;
}
h5 tt,
h5 code {
    font-size: inherit;
}
h6 tt,
h6 code {
    font-size: inherit;
}
h1 {
    font-size: 2.5em;
    line-height: 1.2;

}
h2 {
    font-size: 2em;
    line-height: 1.225;

}

/*@media print {
    .typora-export h1,
    .typora-export h2 {
        border-bottom: none;
        padding-bottom: initial;
    }

    .typora-export h1::after,
    .typora-export h2::after {
        content: "";
        display: block;
        height: 100px;
        margin-top: -96px;
        border-top: 1px solid #eee;
    }
}*/

h3 {
    font-size: 1.5em;
    line-height: 1.43;
}
h4 {
    font-size: 1.25em;
}
h5 {
    font-size: 1em;
}
h6 {
   font-size: 1em;
    color: #777;
}
p,
blockquote,
ul,
ol,
dl,
table{
    margin: 0.8em 0;
}
li>ol,
li>ul {
    margin: 0 0;
}
hr {
    height: 2px;
    padding: 0;
    margin: 16px 0;
    background-color: #e7e7e7;
    border: 0 none;
    overflow: hidden;
    box-sizing: content-box;
}

li p.first {
    display: inline-block;
}
ul,
ol {
    padding-left: 30px;
}
ul:first-child,
ol:first-child {
    margin-top: 0;
}
ul:last-child,
ol:last-child {
    margin-bottom: 0;
}
blockquote {
    border-left: 4px solid #dfe2e5;
    padding: 0 15px;
    color: #777777;
}
blockquote blockquote {
    padding-right: 0;
}
table {
    padding: 0;
    word-break: initial;
}
table tr {
    border: 1px solid #dfe2e5;
    margin: 0;
    padding: 0;
}
table tr:nth-child(2n),
thead {
    background-color: #f8f8f8;
}
table th {
    font-weight: bold;
    border: 1px solid #dfe2e5;
    border-bottom: 0;
    margin: 0;
    padding: 6px 13px;
}
table td {
    border: 1px solid #dfe2e5;
    margin: 0;
    padding: 6px 13px;
}
table th:first-child,
table td:first-child {
    margin-top: 0;
}
table th:last-child,
table td:last-child {
    margin-bottom: 0;
}

.CodeMirror-lines {
    padding-left: 4px;
}

.code-tooltip {
    box-shadow: 0 1px 1px 0 rgba(0,28,36,.3);
    border-top: 1px solid #eef2f2;
}

.md-fences,
code,
tt {
    border: 1px solid #e7eaed;
    background-color: #f8f8f8;
    border-radius: 3px;
    padding: 0;
    padding: 2px 4px 0px 4px;
    font-size: 0.9em;
}

code {
    background-color: #f3f4f4;
    padding: 0 2px 0 2px;
}

.md-fences {
    margin-bottom: 15px;
    margin-top: 15px;
    padding-top: 8px;
    padding-bottom: 6px;
}


.md-task-list-item > input {
  margin-left: -1.3em;
}

@media print {
    html {
        font-size: 13px;
    }
    table,
    pre {
        page-break-inside: avoid;
    }
    pre {
        word-wrap: break-word;
    }
}

.md-fences {
	background-color: #f8f8f8;
}
#write pre.md-meta-block {
	padding: 1rem;
    font-size: 85%;
    line-height: 1.45;
    background-color: #f7f7f7;
    border: 0;
    border-radius: 3px;
    color: #777777;
    margin-top: 0 !important;
}

.mathjax-block>.code-tooltip {
	bottom: .375rem;
}

.md-mathjax-midline {
    background: #fafafa;
}

#write>h3.md-focus:before{
	left: -1.5625rem;
	top: .375rem;
}
#write>h4.md-focus:before{
	left: -1.5625rem;
	top: .285714286rem;
}
#write>h5.md-focus:before{
	left: -1.5625rem;
	top: .285714286rem;
}
#write>h6.md-focus:before{
	left: -1.5625rem;
	top: .285714286rem;
}
.md-image>.md-meta {
    /*border: 1px solid #ddd;*/
    border-radius: 3px;
    padding: 2px 0px 0px 4px;
    font-size: 0.9em;
    color: inherit;
}

.md-tag {
    color: #a7a7a7;
    opacity: 1;
}

.md-toc { 
    margin-top:20px;
    padding-bottom:20px;
}

.sidebar-tabs {
    border-bottom: none;
}

#typora-quick-open {
    border: 1px solid #ddd;
    background-color: #f8f8f8;
}

#typora-quick-open-item {
    background-color: #FAFAFA;
    border-color: #FEFEFE #e5e5e5 #e5e5e5 #eee;
    border-style: solid;
    border-width: 1px;
}

/** focus mode */
.on-focus-mode blockquote {
    border-left-color: rgba(85, 85, 85, 0.12);
}

header, .context-menu, .megamenu-content, footer{
    font-family: "Segoe UI", "Arial", sans-serif;
}

.file-node-content:hover .file-node-icon,
.file-node-content:hover .file-node-open-state{
    visibility: visible;
}

.mac-seamless-mode #typora-sidebar {
    background-color: #fafafa;
    background-color: var(--side-bar-bg-color);
}

.md-lang {
    color: #b4654d;
}

/*.html-for-mac {
    --item-hover-bg-color: #E6F0FE;
}*/

#md-notification .btn {
    border: 0;
}

.dropdown-menu .divider {
    border-color: #e5e5e5;
    opacity: 0.4;
}

.ty-preferences .window-content {
    background-color: #fafafa;
}

.ty-preferences .nav-group-item.active {
    color: white;
    background: #999;
}

.menu-item-container a.menu-style-btn {
    background-color: #f5f8fa;
    background-image: linear-gradient( 180deg , hsla(0, 0%, 100%, 0.8), hsla(0, 0%, 100%, 0)); 
}



</style>

</head>



<main class='container col-11'>


<div class="my-3 p-3 bg-body rounded shadow-lg row" >



<div class='typora-export os-windows typora-export-show-outline typora-export-no-collapse-outline'><div class='typora-export-content'>
<div class="typora-export-sidebar"><div class="outline-content"><li class="outline-item-wrapper outline-h2"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#31amino-acids-and-proteins"><strong>3.1.Amino Acids and Proteins</strong></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h2"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#32physical-properties-of-amino-acids"><strong>3.2.Physical Properties of Amino Acids</strong></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h2"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#33classification-of-amino-acids"><strong>3.3.Classification of Amino Acids</strong></a></div><ul class="outline-children"><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#331according-to-the-different-classification-of-r-groups">3.3.1.According to the different classification of R groups</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#332according-to-the-nature-of-the-r-group">3.3.2.According to the nature of the R group</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#333according-to-the-degree-of-needs-of-the-body">3.3.3.According to the degree of needs of the body</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#334according-to-whether-the-protein-is-constituted-or-not-it-is-not-distinguished">3.3.4.According to whether the protein is constituted or not, it is not distinguished</a></div><ul class="outline-children"></ul></li></ul></li><li class="outline-item-wrapper outline-h2"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#34amphoteric-dissociation-of-amino-acids">3.4.Amphoteric dissociation of amino acids</a></div><ul class="outline-children"><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#341acid-base-behavior-of-amino-acids">3.4.1.Acid-base behavior of amino acids</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#342titration-curves-of-amino-acids">3.4.2.Titration curves of amino acids</a></div><ul class="outline-children"><li class="outline-item-wrapper outline-h4"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#ⅰ-titration-curve-of-r-based-non-dissociated-amino-acids"><strong>(Ⅰ) Titration curve of R-based non-dissociated amino acids</strong></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h4"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#ⅱ-titration-curves-of-r-based-dissociable-amino-acids"><strong>(Ⅱ) Titration curves of R-based dissociable amino acids</strong></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h4"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#ⅲ-calculation-of-isoelectric-points">(Ⅲ) Calculation of isoelectric points</a></div><ul class="outline-children"></ul></li></ul></li></ul></li><li class="outline-item-wrapper outline-h2"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#35chemical-reactions-of-amino-acids">3.5.Chemical reactions of amino acids</a></div><ul class="outline-children"><li class="outline-item-wrapper outline-h3 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#351reactions-involving-α-nh2">3.5.1.Reactions involving α-NH<sub>2</sub></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#352reactions-involving-α-cooh">3.5.2.Reactions involving α-COOH</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#353reactions-involving-α-nh2-and-α-cooh">3.5.3.Reactions involving α-NH<sub>2</sub> and α-COOH</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#354reactions-involving-r-groups">3.5.4.Reactions involving R groups</a></div><ul class="outline-children"></ul></li></ul></li><li class="outline-item-wrapper outline-h2 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#36peptides-and-peptide-bonds">3.6.Peptides and Peptide Bonds</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h2 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#37other-molecular-forces">3.7.Other Molecular Forces</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h2 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#38active-peptides-and-polypeptides"><strong>3.8.Active peptides and Polypeptides</strong></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h2 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#39basic-characteristics-of-proteins"><strong>3.9.Basic Characteristics of Proteins</strong></a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h2 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#310composition-and-classification-of-proteins"><strong>3.10.Composition and Classification of Proteins</strong></a></div><ul class="outline-children"><li class="outline-item-wrapper outline-h3 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#3101classification-based-on-shape">3.10.1.Classification based on shape</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3 outline-item-open"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#3102classification-based-on-composition">3.10.2.Classification based on composition</a></div><ul class="outline-children"></ul></li><li class="outline-item-wrapper outline-h3"><div class="outline-item"><span class="outline-expander"></span><a class="outline-label" href="#3103classification-based-on-structure">3.10.3.Classification based on structure</a></div><ul class="outline-children"></ul></li></ul></li>
<br><span style='margin-left:25px; font-size:16px;' >Previous</span><br>
<a style='margin-left:35px;  font-size:10px;' href='2.Biomolecules.html'>2.Biomolecules</a>
<br><span style='margin-left:25px; font-size:16px;'>Next</span><br>
<a style='margin-left:35px;  font-size:10px;' href='4.Protein Structure and Function.html'>4.Protein Structure and Function</a>
<br><br>

</div></div>
    <div id='write'  class=''><div style="background-color:whitesmoke">
<details style='font-size:12px;'>
    <summary><strong>3.Amino acids and peptides</strong></summary>
    <a href="index.html">Return to Introduction</a><br><hr>
    <strong>Previous</strong>:<a href="2.Biomolecules.html">2.Biomolecules</a><br>
    <strong>Next</strong>:<a href="4.Protein Structure and Function.html">4.Protein Structure and Function</a>
</details>
</div><h2 id='31amino-acids-and-proteins'><strong><span>3.1.Amino Acids and Proteins</span></strong></h2><p><strong><span>Protein</span><span>	</span></strong><span>refers to a class of complex organic macromolecular compounds that are essential for maintaining life activities; It is composed of 20 kinds of α amino acids through dehydration and condensation, which can contain one or more peptide chains, and has functional diversity and structural stability.</span></p><p><img src="media/3/蛋白综述.jpg"  /></p><p><span>Protein is one of the most abundant and important biological macromolecules in living systems, and is abundant in prokaryotic cells, eukaryotic cells, cell walls, cell membranes, and organelles. The types, structures, and functions of proteins vary from oligopeptides ，such as oxytocin (9 peptides),  to multi-subunit polymers. The simple peptide chain structure (primary structure) is the structural basis of the protein, and then rises to the spacious level (secondary, tertiary, and quaternary structure) to form the complex protein higher order structure through non-covalent forces.</span></p><p><span>The basic structural components that make up proteins (peptide chains) are amino acids. </span><strong><span>There are 20 kinds of amino acids in the common protein</span></strong><span>, and of course there are rare amino acids</span><sup class='md-footnote'><a href='#dfref-footnote-1' name='ref-footnote-1'>1</a></sup><span>, non-protein amino acids</span><sup class='md-footnote'><a href='#dfref-footnote-2' name='ref-footnote-2'>2</a></sup><span>. Through different permutations and combinations, amino acids form proteins with different biological functions, such as enzymes (catalytic function), carrier proteins (transport function), antibodies (immune function), peptide hormones (regulatory function), and so on</span></p><p><span> </span></p><p><span> </span></p><h2 id='32physical-properties-of-amino-acids'><strong><span>3.2.Physical Properties of Amino Acids</span></strong></h2><p><img src="media/3/氨基酸分子.jpg" style="zoom: 50%;" /></p><p><strong><span>Amino acid</span></strong><span>	</span><span>refers to a class that contains both -COOH and -NH</span><sub><span>2</span></sub><span> organic molecules are collectively referred to. Proteins (polypeptide chains) are polymers formed by dehydration and condensation between amino acids, and adjacent amino acids are connected by peptide bonds.</span></p><p><span>It is worth mentioning that in old textbooks, there are 20 types of amino acids that make up proteins. In fact, it has been discovered that </span><strong><span>selenocysteine</span></strong><span> and </span><strong><span>pyrroletyrosine</span></strong><span> are the 21st and 22nd natural protein amino acids.</span></p><figure><table><thead><tr><th style='text-align:center;' ><span>abbreviation</span></th><th style='text-align:center;' ><span>Chinese</span></th><th><span>PK1(α-COOH)</span></th><th><span>PK2(α-NH2)</span></th><th><span>PKR</span></th><th><span>PI</span></th></tr></thead><tbody><tr><td style='text-align:center;' ><strong><span>Polar negatively charged amino acids</span></strong></td><td style='text-align:center;' >&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td style='text-align:center;' ><span>Asp（D）</span></td><td style='text-align:center;' ><span>aspartic acid</span></td><td><span>1.88</span></td><td><span>9.60</span></td><td><strong><span>3.65（β-COOH）</span></strong></td><td><strong><span>2.77</span></strong></td></tr><tr><td style='text-align:center;' ><span>Glu（E）</span></td><td style='text-align:center;' ><span>glutamic acid</span></td><td><span>2.19</span></td><td><span>9.67</span></td><td><strong><span>4.25（γ-COOH）</span></strong></td><td><strong><span>3.22</span></strong></td></tr><tr><td style='text-align:center;' ><strong><span>Polar positively charged amino acids</span></strong></td><td style='text-align:center;' >&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td style='text-align:center;' ><span>Arg（R）</span></td><td style='text-align:center;' ><span>arginine</span></td><td><span>2.17</span></td><td><span>9.04</span></td><td><strong><span>12.48(Guanidine)</span></strong></td><td><strong><span>10.76</span></strong></td></tr><tr><td style='text-align:center;' ><span>Lys（K）</span></td><td style='text-align:center;' ><span>lysine</span></td><td><span>2.18</span></td><td><span>8.95</span></td><td><strong><span>10.53（ε-NH2）</span></strong></td><td><strong><span>9.47</span></strong></td></tr><tr><td style='text-align:center;' ><span>His（H）</span></td><td style='text-align:center;' ><span>histidine</span></td><td><span>1.82</span></td><td><span>9.17</span></td><td><strong><span>6.00 (imidazolyl)</span></strong></td><td><strong><span>7.59</span></strong></td></tr><tr><td style='text-align:center;' ><strong><span>Polar uncharged amino acids</span></strong></td><td style='text-align:center;' >&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td style='text-align:center;' ><span>Asn（N）</span></td><td style='text-align:center;' ><span>asparagine</span></td><td><span>2.02</span></td><td><span>8.80</span></td><td>&nbsp;</td><td><span>5.41</span></td></tr><tr><td style='text-align:center;' ><span>Gln（Q）</span></td><td style='text-align:center;' ><span>glutamine</span></td><td><span>2.17</span></td><td><span>9.13</span></td><td>&nbsp;</td><td><span>5.65</span></td></tr><tr><td style='text-align:center;' ><span>Thr（T）</span></td><td style='text-align:center;' ><span>threonine</span></td><td><span>2.11</span></td><td><span>9.62</span></td><td>&nbsp;</td><td><span>5.87</span></td></tr><tr><td style='text-align:center;' ><span>Ser（S）</span></td><td style='text-align:center;' ><span>serine</span></td><td><span>2.21</span></td><td><span>9.15</span></td><td>&nbsp;</td><td><span>5.68</span></td></tr><tr><td style='text-align:center;' ><span>Tyr（Y）</span></td><td style='text-align:center;' ><span>tyrosine</span></td><td><span>2.20</span></td><td><span>9.11</span></td><td><strong><span>10.07 (phenol-OH)</span></strong></td><td><span>5.66</span></td></tr><tr><td style='text-align:center;' ><span>Cys（C）</span></td><td style='text-align:center;' ><span>cysteine</span></td><td><span>1.96</span></td><td><span>10.28</span></td><td><strong><span>8.18(Mercapto-SH)</span></strong></td><td><span>5.07</span></td></tr><tr><td style='text-align:center;' ><strong><span>Non-polar uncharged amino acids</span></strong></td><td style='text-align:center;' >&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td><span>5.89</span></td></tr><tr><td style='text-align:center;' ><span>Leu（L）</span></td><td style='text-align:center;' ><span>leucine</span></td><td><span>2.36</span></td><td><span>9.60</span></td><td>&nbsp;</td><td><span>5.98</span></td></tr><tr><td style='text-align:center;' ><span>Ile（I）</span></td><td style='text-align:center;' ><span>isoleucine</span></td><td><span>2.36</span></td><td><span>9.68</span></td><td>&nbsp;</td><td><span>6.02</span></td></tr><tr><td style='text-align:center;' ><span>Val（V）</span></td><td style='text-align:center;' ><span>valine</span></td><td><span>2.32</span></td><td><span>9.62</span></td><td>&nbsp;</td><td><span>5.97</span></td></tr><tr><td style='text-align:center;' ><span>Phe（F）</span></td><td style='text-align:center;' ><span>phenylalanine</span></td><td><span>1.83</span></td><td><span>9.13</span></td><td>&nbsp;</td><td><span>5.48</span></td></tr><tr><td style='text-align:center;' ><span>Trp（W）</span></td><td style='text-align:center;' ><span>tryptophan</span></td><td><span>2.38</span></td><td><span>9.39</span></td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td style='text-align:center;' ><span>Met（M）</span></td><td style='text-align:center;' ><span>methionine</span></td><td><span>2.28</span></td><td><span>9.21</span></td><td>&nbsp;</td><td><span>5.75</span></td></tr><tr><td style='text-align:center;' ><span>Pro（P）</span></td><td style='text-align:center;' ><span>proline</span></td><td><span>1.99</span></td><td><span>10.96</span></td><td>&nbsp;</td><td><span>6.48</span></td></tr><tr><td style='text-align:center;' ><strong><span>Somewhere between polar and non-polar</span></strong></td><td style='text-align:center;' >&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td style='text-align:center;' ><span>Gly（G）</span></td><td style='text-align:center;' ><span>glycine</span></td><td><span>2.34</span></td><td><span>9.60</span></td><td>&nbsp;</td><td><span>5.97</span></td></tr><tr><td style='text-align:center;' ><span>Ala（A）</span></td><td style='text-align:center;' ><span>alanine</span></td><td><span>2.34</span></td><td><span>9.69</span></td><td>&nbsp;</td><td><span>6.02</span></td></tr></tbody></table></figure><p>&nbsp;</p><p><span>The 20 amino acids found in natural proteins are all α-amino acids, which </span><strong><span>α</span></strong><span> means that the free amino group and the free carboxyl group are linked to a C</span><sub><span>α</span></sub><span>. C</span><sub><span>α</span></sub><span> There is also an H and an R group (the position of the R group of </span><strong><span>Gly is an H),</span></strong><span> and the size, structure, and electrical properties of the R group largely determine the different properties between amino acids.</span></p><p><img src="media/3/氨基酸中的碳名称.jpg" style="zoom: 50%;" /></p><p><span>As mentioned above, the R group of Gly is an H, so all natural proteins have chiral carbon C* except </span><strong><span>Gly</span></strong><sup class='md-footnote'><a href='#dfref-footnote-3' name='ref-footnote-3'>3</a></sup><span>, all of which are L-type (</span><strong><span>L、D-type</span></strong><span> and left-handed and right-handed have no relationship), and some D-type amino acids are present in some antibiotics and short peptide tail of peptidoglycan in the cell wall of bacteria</span></p><p><span>Amino acids are small molecules with an average molecular weight of 128 Da, but the average molecular weight of amino acid residues in proteins is </span><strong><span>110 Da</span></strong><span> due to the dehydration and condensation of synthetic peptide chains</span></p><p><span>Amino acids are colorless crystals with high melting points and are mostly zwitterionic in solution or crystals</span></p><p><span>Soluble in water, dilute acid, dilute alkali, insoluble in organic solvents, so it can be extracted through </span><strong><span>ethanol</span></strong><span> precipitation (the same is true for nucleic acid extraction).</span></p><p><span>Different amino acids have different tastes, for example, monosodium glutamate is monosodium glutamate, so glutamic acid has umami taste</span></p><p><span> </span></p><p><span> </span></p><h2 id='33classification-of-amino-acids'><strong><span>3.3.Classification of Amino Acids</span></strong></h2><p><img src="media/3/氨基酸快速查询.png" style="zoom: 100%;" /></p><h3 id='331according-to-the-different-classification-of-r-groups'><span>3.3.1.According to the different classification of R groups</span></h3><p><strong><span>(1) Aromatic amino acids</span></strong></p><p><img src="media/3/紫外吸收.jpg" style="zoom: 40%;" /></p><p><strong><span>Phe, Tyr, Trp</span></strong><span> all have aromatic side chains</span></p><p><strong><span>Ultraviolet absorption:</span></strong><span> Among the 20 protein amino acids, only aromatic amino acids have ultraviolet absorption, where Phe has the maximum OD</span><sub><span>257</span></sub><span>, Tyr has the maximum OD</span><sub><span>275</span></sub><span>, Trp has the maximum OD</span><sub><span>280</span></sub><span>。 The light absorption of Phe is smaller than that of Tyr and Trp, so it does not contribute much to the light absorption of proteins</span></p><p><span>Because most proteins contain the above three amino acids, proteins have the property of ultraviolet light absorption at 280nm, which is an important method for protein quantitative detection (colorimetric method)</span></p><p>&nbsp;</p><p><strong><span>(2) Heterocyclic amino acids</span></strong></p><p><strong><span>His</span></strong><span> and </span><strong><span>Pro</span></strong><span> are heterocyclic amino acids, where Pro is a heterocyclic amino acid</span></p><p>&nbsp;</p><p><strong><span>(3) Aliphatic amino acids</span></strong></p><p><span>The other 15 amino acids</span></p><p><span> </span></p><h3 id='332according-to-the-nature-of-the-r-group'><span>3.3.2.According to the nature of the R group</span></h3><div class="md-diagram-panel md-fences-adv-panel"><svg id="mermaidChart0" width="100%" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" height="348.4000244140625" style="max-width: 623.7281494140625px;" viewBox="0 0.0000019073486328125 623.7281494140625 348.4000244140625"><style>#mermaidChart0 {font-family:sans-serif;font-size:16px;fill:#333;}#mermaidChart0 .error-icon{fill:#552222;}#mermaidChart0 .error-text{fill:#552222;stroke:#552222;}#mermaidChart0 .edge-thickness-normal{stroke-width:2px;}#mermaidChart0 .edge-thickness-thick{stroke-width:3.5px;}#mermaidChart0 .edge-pattern-solid{stroke-dasharray:0;}#mermaidChart0 .edge-pattern-dashed{stroke-dasharray:3;}#mermaidChart0 .edge-pattern-dotted{stroke-dasharray:2;}#mermaidChart0 .marker{fill:#333333;stroke:#333333;}#mermaidChart0 .marker.cross{stroke:#333333;}#mermaidChart0 svg{font-family:sans-serif;font-size:16px;}#mermaidChart0 .label{font-family:sans-serif;color:#333;}#mermaidChart0 .cluster-label text{fill:#333;}#mermaidChart0 .cluster-label span{color:#333;}#mermaidChart0 .label text,#mermaidChart0 span{fill:#333;color:#333;}#mermaidChart0 .node rect,#mermaidChart0 .node circle,#mermaidChart0 .node ellipse,#mermaidChart0 .node polygon,#mermaidChart0 .node path{fill:#ECECFF;stroke:#9370DB;stroke-width:1px;}#mermaidChart0 .node .label{text-align:center;}#mermaidChart0 .node.clickable{cursor:pointer;}#mermaidChart0 .arrowheadPath{fill:#333333;}#mermaidChart0 .edgePath .path{stroke:#333333;stroke-width:2.0px;}#mermaidChart0 .flowchart-link{stroke:#333333;fill:none;}#mermaidChart0 .edgeLabel{background-color:#e8e8e8;text-align:center;}#mermaidChart0 .edgeLabel rect{opacity:0.5;background-color:#e8e8e8;fill:#e8e8e8;}#mermaidChart0 .cluster rect{fill:#ffffde;stroke:#aaaa33;stroke-width:1px;}#mermaidChart0 .cluster text{fill:#333;}#mermaidChart0 .cluster span{color:#333;}#mermaidChart0 div.mermaidTooltip{position:absolute;text-align:center;max-width:200px;padding:2px;font-family:sans-serif;font-size:12px;background:hsl(80, 100%, 96.2745098039%);border:1px solid #aaaa33;border-radius:2px;pointer-events:none;z-index:100;}#mermaidChart0 :root{--mermaid-font-family:sans-serif;}#mermaidChart0 :root{--mermaid-alt-font-family:sans-serif;}</style><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath LS-Aa LE-Positively_Charged_Polar" id="L-Aa-Positively_Charged_Polar" style="opacity: 1;"><path class="path" d="M35.220004662629535,151.39999771118164L73.11249923706055,30.799999237060547L103.20000076293945,30.799999237060547" marker-end="url(#arrowhead18)" style="fill:none"></path><defs><marker id="arrowhead18" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Positively_Charged_Polar LE-Arg,His,Lys" id="L-Positively_Charged_Polar-Arg,His,Lys" style="opacity: 1;"><path class="path" d="M312.21249771118164,30.799999237060547L342.29999923706055,30.799999237060547L440.23281478881836,30.799999237060547" marker-end="url(#arrowhead19)" style="fill:none"></path><defs><marker id="arrowhead19" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Aa LE-Negatively_Charged_Polar" id="L-Aa-Negatively_Charged_Polar" style="opacity: 1;"><path class="path" d="M48.11249923706055,152.92240006477314L73.11249923706055,126.39999771118164L98.11249923706055,126.39999771118164" marker-end="url(#arrowhead20)" style="fill:none"></path><defs><marker id="arrowhead20" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Negatively_Charged_Polar LE-Glu,Asp" id="L-Negatively_Charged_Polar-Glu,Asp" style="opacity: 1;"><path class="path" d="M317.29999923706055,126.39999771118164L342.29999923706055,126.39999771118164L452.13906478881836,126.39999771118164" marker-end="url(#arrowhead21)" style="fill:none"></path><defs><marker id="arrowhead21" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Aa LE-Uncharged_Polar" id="L-Aa-Uncharged_Polar" style="opacity: 1;"><path class="path" d="M48.11249923706055,195.47759383171123L73.11249923706055,221.99999618530273L132.87499618530273,221.99999618530273" marker-end="url(#arrowhead22)" style="fill:none"></path><defs><marker id="arrowhead22" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Uncharged_Polar LE-Thr,Tyr,Ser,Cys,Asn,Gln,Gly" id="L-Uncharged_Polar-Thr,Tyr,Ser,Cys,Asn,Gln,Gly" style="opacity: 1;"><path class="path" d="M282.53750228881836,221.99999618530273L342.29999923706055,221.99999618530273L382.9156303405762,221.99999618530273" marker-end="url(#arrowhead23)" style="fill:none"></path><defs><marker id="arrowhead23" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Aa LE-Nonpolar" id="L-Aa-Nonpolar" style="opacity: 1;"><path class="path" d="M35.220004662629535,196.99999618530273L73.11249923706055,317.5999946594238L161.3812484741211,317.5999946594238" marker-end="url(#arrowhead24)" style="fill:none"></path><defs><marker id="arrowhead24" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Nonpolar LE-Leu,Ile,Phe,Val,Trp,Met,Pro,Ala" id="L-Nonpolar-Leu,Ile,Phe,Val,Trp,Met,Pro,Ala" style="opacity: 1;"><path class="path" d="M254.03125,317.5999946594238L342.29999923706055,317.5999946594238L367.29999923706055,317.5999946594238" marker-end="url(#arrowhead25)" style="fill:none"></path><defs><marker id="arrowhead25" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Aa-Positively_Charged_Polar" class="edgeLabel L-LS-Aa' L-LE-Positively_Charged_Polar"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Positively_Charged_Polar-Arg,His,Lys" class="edgeLabel L-LS-Positively_Charged_Polar' L-LE-Arg,His,Lys"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Aa-Negatively_Charged_Polar" class="edgeLabel L-LS-Aa' L-LE-Negatively_Charged_Polar"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Negatively_Charged_Polar-Glu,Asp" class="edgeLabel L-LS-Negatively_Charged_Polar' L-LE-Glu,Asp"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Aa-Uncharged_Polar" class="edgeLabel L-LS-Aa' L-LE-Uncharged_Polar"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Uncharged_Polar-Thr,Tyr,Ser,Cys,Asn,Gln,Gly" class="edgeLabel L-LS-Uncharged_Polar' L-LE-Thr,Tyr,Ser,Cys,Asn,Gln,Gly"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Aa-Nonpolar" class="edgeLabel L-LS-Aa' L-LE-Nonpolar"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Nonpolar-Leu,Ile,Phe,Val,Trp,Met,Pro,Ala" class="edgeLabel L-LS-Nonpolar' L-LE-Leu,Ile,Phe,Val,Trp,Met,Pro,Ala"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node default" id="flowchart-Aa-12" transform="translate(28.056249618530273,174.1999969482422)" style="opacity: 1;"><rect rx="0" ry="0" x="-20.05625057220459" y="-22.800000190734863" width="40.11250114440918" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-10.05625057220459,-12.800000190734863)"><foreignObject width="20.11250114440918" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Aa</div></foreignObject></g></g></g><g class="node default" id="flowchart-Positively_Charged_Polar-13" transform="translate(207.70624923706055,30.799999237060547)" style="opacity: 1;"><rect rx="0" ry="0" x="-104.5062484741211" y="-22.800000190734863" width="209.0124969482422" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-94.5062484741211,-12.800000190734863)"><foreignObject width="189.0124969482422" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Positively_Charged_Polar</div></foreignObject></g></g></g><g class="node default" id="flowchart-Arg,His,Lys-14" transform="translate(491.51406478881836,30.799999237060547)" style="opacity: 1;"><rect rx="0" ry="0" x="-51.28125" y="-22.800000190734863" width="102.5625" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-41.28125,-12.800000190734863)"><foreignObject width="82.5625" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Arg,His,Lys</div></foreignObject></g></g></g><g class="node default" id="flowchart-Negatively_Charged_Polar-16" transform="translate(207.70624923706055,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-109.59375" y="-22.800000190734863" width="219.1875" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-99.59375,-12.800000190734863)"><foreignObject width="199.1875" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Negatively_Charged_Polar</div></foreignObject></g></g></g><g class="node default" id="flowchart-Glu,Asp-17" transform="translate(491.51406478881836,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-39.375" y="-22.800000190734863" width="78.75" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-29.375,-12.800000190734863)"><foreignObject width="58.75" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Glu,Asp</div></foreignObject></g></g></g><g class="node default" id="flowchart-Uncharged_Polar-19" transform="translate(207.70624923706055,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-74.83125305175781" y="-22.800000190734863" width="149.66250610351562" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-64.83125305175781,-12.800000190734863)"><foreignObject width="129.66250610351562" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Uncharged_Polar</div></foreignObject></g></g></g><g class="node default" id="flowchart-Thr,Tyr,Ser,Cys,Asn,Gln,Gly-20" transform="translate(491.51406478881836,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-108.59844207763672" y="-22.800000190734863" width="217.19688415527344" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-98.59844207763672,-12.800000190734863)"><foreignObject width="197.19688415527344" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Thr,Tyr,Ser,Cys,Asn,Gln,Gly</div></foreignObject></g></g></g><g class="node default" id="flowchart-Nonpolar-22" transform="translate(207.70624923706055,317.5999946594238)" style="opacity: 1;"><rect rx="0" ry="0" x="-46.32500076293945" y="-22.800000190734863" width="92.6500015258789" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-36.32500076293945,-12.800000190734863)"><foreignObject width="72.6500015258789" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Nonpolar</div></foreignObject></g></g></g><g class="node default" id="flowchart-Leu,Ile,Phe,Val,Trp,Met,Pro,Ala-23" transform="translate(491.51406478881836,317.5999946594238)" style="opacity: 1;"><rect rx="0" ry="0" x="-124.21406555175781" y="-22.800000190734863" width="248.42813110351562" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-114.21406555175781,-12.800000190734863)"><foreignObject width="228.42813110351562" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Leu,Ile,Phe,Val,Trp,Met,Pro,Ala</div></foreignObject></g></g></g></g></g></g></svg></div><p>&nbsp;</p><blockquote><p><span>Polarity = Hydrophilic = Soluble in water</span></p><p><span>Non-polar = hydrophobic = insoluble in water</span></p></blockquote><p><strong><span>(1) Polar negatively charged amino acids</span></strong></p><p><strong><span>Acidic amino acids (acidic aa)</span></strong><span>	</span><span>refers to molecules containing one amino group and two carboxyl groups at the same time, pI is much smaller than physiological pH, and the physiological pH is negatively charged</span></p><p><span>The second carboxyl group of Asp is </span><strong><span>β-COOH</span></strong><span> and the second carboxyl group of Glu is </span><strong><span>γ-COOH</span></strong></p><p>&nbsp;</p><p><strong><span>(2) Polar positively charged amino acids</span></strong></p><p><strong><span>Basic amino acids (basic aa)</span></strong><span> </span><span>	</span><span>refers to molecules containing both a carboxyl group and multiple amino groups in the molecule, pI is much larger than the physiological pH, and the physiological pH is positively charged</span></p><p><strong><span>Arg</span></strong><span> contains two amino groups, two imino groups, and the guanidine group is positively charged; </span><strong><span>Lys</span></strong><span> contains two amino groups, the other being </span><strong><span>ε-NH</span><sub><span>2</span></sub></strong><span>； </span><strong><span>His</span></strong><span> contains one amino group and one imino group, and the imidazole group is positively charged</span></p><p>&nbsp;</p><p><strong><span>(3) Polar uncharged amino acids</span></strong></p><p><span>The polarity of </span><strong><span>Ser, Thr, Tyr</span></strong><span> is provided by -OH</span></p><p><span>The polarity of </span><strong><span>Asn</span></strong><span> and </span><strong><span>Gln</span></strong><span> is provided by the amide group; Asn and Gln are easily acidolyzed or alkalinized to synthesize Asp and Glu, respectively</span></p><p><span>The polarity of </span><strong><span>Cys</span></strong><span> is provided by sulfhydryl(-SH), and the two Cys are connected by -SH to form </span><strong><span>disulfide bonds</span></strong><span>, which form </span><strong><span>cystine</span></strong><span>; Disulfide bonds play an important role in maintaining protein conformation, and the more disulfide bonds, the more stable the protein structure</span></p><p>&nbsp;</p><p><strong><span>(4) Non-polar amino acids</span></strong></p><p><span>The R groups of these amino acids are all non-polar (hydrophobic).</span></p><p><span>The polarity of </span><strong><span>Ala</span></strong><span> and </span><strong><span>Gly</span></strong><span> is small enough to be between polar and nonpolar</span></p><p><strong><span>Ile, Leu, and Val</span></strong><span> are collectively known as </span><strong><span>branched-chain amino acids</span></strong></p><p><strong><span>Phe and Trp</span></strong><span> belong to the aromatic amino acids</span></p><p><strong><span>Met</span></strong><span> is a sulfur-containing amino acid, and another sulfur-containing amino acid is Cys</span></p><p><span> </span></p><h3 id='333according-to-the-degree-of-needs-of-the-body'><span>3.3.3.According to the degree of needs of the body</span></h3><p><strong><span>(1) Essential amino acids</span></strong></p><p><strong><span>Essential amino acids</span></strong><span> that are necessary for the body&#39;s growth and metabolism, but need to be obtained from food because they cannot be synthesized by themselves.</span></p><ul><li><span>Ile、Met、Val、Leu、Trp、Phe、Thr、Lys</span></li></ul><p>&nbsp;</p><p><strong><span>(2) Semi-essential amino acids</span></strong></p><p><strong><span>Semi-essential amino acids</span></strong><span> that cannot be synthesized by the zombie body only under special circumstances and need to be obtained from food.</span></p><ul><li><span>Arg、His</span></li></ul><p>&nbsp;</p><p><strong><span>(3) Non-essential amino acids</span></strong></p><p><strong><span>Non-essential amino acids </span></strong><span> can be metabolized and synthesized by the zombie body without additional intake.</span></p><ul><li><span>10 other kinds</span></li></ul><p><span> </span></p><h3 id='334according-to-whether-the-protein-is-constituted-or-not-it-is-not-distinguished'><span>3.3.4.According to whether the protein is constituted or not, it is not distinguished</span></h3><p><strong><span>(1) Standard protein amino acids</span></strong></p><p><span>20 α-amino acids</span></p><p>&nbsp;</p><p><strong><span>(2) Non-protein amino acids</span></strong></p><p><span>It refers to the rare amino acids derived from 20 kinds of standard protein amino acids, most of which are not used as synthetic components of proteins, but participate in other metabolisms of the body in a free or bound state.</span></p><ul><li><span>Hydroxy Pro（HyPro）, Hydroxy Lys（HyLys）, Selenocysteine, Pyrrole Tyrosine, Ornithine（Orn）, Citrulline（Cit）, D-Alanine, Taurine, Theanine, β-Alanine, etc.</span></li><li><span>Among them</span><strong><span>, ornithine</span></strong><span> and </span><strong><span>citrulline</span></strong><span> are important intermediates in the urea cycle.</span></li></ul><p><span> </span></p><p><span> </span></p><h2 id='34amphoteric-dissociation-of-amino-acids'><span>3.4.Amphoteric dissociation of amino acids</span></h2><h3 id='341acid-base-behavior-of-amino-acids'><span>3.4.1.Acid-base behavior of amino acids</span></h3><p><strong><span>Amphoteric dissociation of amino acids</span></strong><span>	</span><span>amino acid molecules contain dissociable α-amino groups, α-carboxyl groups and partially dissociable R groups, so dissociation occurs in different pH solutions, which can make amino acids net positively charged, negatively charged, or in a facultative ionic state with a net charge of 0. Amino acids are amphoteric molecules, so proteins are also amphoteric molecules. </span></p><p><strong><span>Dipolarion</span></strong><span>	</span><span>refers to a molecule with amphoteric dissociation attributes, when the degree of dissociation of positive and negative charges is the same, the net charge is 0, the molecule is in a state, at this time the molecule can be used as both a proton donor and a proton acceptor.</span></p><p><img src="media/3/氨基酸的两性解离.jpg" alt="img" style="zoom: 67%;" /></p><p><strong><span>Isoelectric point</span></strong><span>	</span><span>refers to the pH of the solution when the amphoteric molecules dissociate the positive and negative charges to the same extent in a certain pH solution, that is, when the net charge is 0, the solubility of the molecules is the smallest and does not move in the electric field. When the pH of the solution is much smaller than the pI, the amino acids are fully protonated, and the -COOH and -NH</span><sub><span>3</span></sub><sup><span>+</span></sup><span> after complete protonation can release protons; When the pH of the solution is much greater than the pI, the amino acids are completely deprotonated, at which -COO</span><sup><span>-</span></sup><span> and -NH</span><sub><span>2</span></sub><span> can accept protons.</span></p><p><span> </span></p><p><span> </span></p><h3 id='342titration-curves-of-amino-acids'><span>3.4.2.Titration curves of amino acids</span></h3><blockquote><p><span>To be clear, the state of complete dissociation of -COOH is -COO</span><sup><span>-</span></sup><span>, α-NH</span><sub><span>3</span></sub><sup><span>+</span></sup><span> is α-NH2</span></p></blockquote><h4 id='ⅰ-titration-curve-of-r-based-non-dissociated-amino-acids'><strong><span>(Ⅰ) Titration curve of R-based non-dissociated amino acids</span></strong></h4><p><img src="media/3/甘氨酸滴定.jpg" alt="img" style="zoom:33%;" /></p><p><span>Acid-base titration is essentially the process of gradually increasing or decreasing protons. The figure shows the titration curve of Gly</span></p><p><span>When the pH is very low, </span><strong><span>The dominant ionic state in Gly is </span><sup><span>+</span></sup><span>H</span><sub><span>3</span></sub><span>N-CH</span><sub><span>2</span></sub><span>-COOH, which is in a fully protonated state with a net charge of +1</span></strong><span>； pH reaches the dissociation equilibrium point of α-COOH pK</span><sub><span>1</span></sub><span>, α-COOH dissociates half, at which point  is  </span><strong><span>the same moore&#39;s </span><sup><span>+</span></sup><span>H</span><sub><span>3</span></sub><span>N-CH</span><sub><span>2</span></sub><span>- COOH and </span><sup><span>+</span></sup><span>H</span><sub><span>3</span></sub><span>N-CH</span><sub><span>2</span></sub><span>- COO</span><sup><span>-</span></sup><span> coexistence state with a net charge of +0.5</span></strong><span>, at which point the curve has its first turn; When the pH continues to rise, equal to pI, the curve has a second turn, and -COOH is completely dissociated, </span><strong><span>the dominant ionic state is </span><sup><span>+</span></sup><span>H</span><sub><span>3</span></sub><span>N-CH</span><sub><span>2</span></sub><span>-COO</span><sup><span>-</span></sup><span>, net charge is 0, reaches pI</span></strong><span>; The pH continues to rise, reaching α-NH</span><sup><span>3+</span></sup><span>, the dissociation equilibrium point pK</span><sub><span>2</span></sub><span>, α-NH</span><sub><span>3+</span></sub><span> dissociate half, at which point  is  </span><strong><span>the same moore&#39;s H</span><sub><span>2</span></sub><span>N-CH</span><sub><span>2</span></sub><span>- COO</span><sup><span>-</span></sup><span> and </span><sup><span>+</span></sup><span>H</span><sub><span>3</span></sub><span>N-CH</span><sub><span>2</span></sub><span>- COO</span><sup><span>-</span></sup><span> coexistence state with a net charge of -0.5</span></strong><span>, at which point there is a third turn in the curve; When the pH is very high,</span><strong><span>The dominant ionic state in Gly is H</span><sub><span>2</span></sub><span>N-CH</span><sub><span>2</span></sub><span>-COO</span><sup><span>-</span></sup><span>, at the state of complete deprotonation, has a net charge of -1</span></strong></p><p><span>The analysis curve shows two regions of flatness, which correspond to the half-dissociation points of α-COOH and α-NH</span><sub><span>2</span></sub><span> groups. At these points, the pH remains relatively stable even with the addition or removal of acid, indicating that the amino acid has buffering capacity. However, at the physiological pH of blood (≈ 7), Gly clearly lacks buffering capacity.</span></p><p><span> </span></p><h4 id='ⅱ-titration-curves-of-r-based-dissociable-amino-acids'><strong><span>(Ⅱ) Titration curves of R-based dissociable amino acids</span></strong></h4><p><img src="media/3/组氨酸滴定.jpg" style="zoom: 33%;" /></p><p><span>The figure shows the titration curve of His, which represents the titration process of a series of R-group dissociable amino acids</span></p><p><span>His contains one amino group and one imino group. When the pH is very low and fully protonated, </span><strong><span>the net charge of His is +2</span></strong><span>. When the pH gradually increases, -COOH is the first to completely dissociate, </span><strong><span>and the net charge is +1</span></strong><span>. Therefore, only the R-group (imidazolyl) is completely dissociated, the pI of His can only be reached before α-NH</span><sup><span>3+</span></sup><span> dissociation begins</span></p><p><span>His contains imidazolyl, imidazolyl&#39;s pK</span><sub><span>R</span></sub><span>(=6.00) is closest to physiological pH, so in the blood of physiological pH, </span><strong><span>His has buffering capacity</span></strong><span>, so there is often an enzyme activity center and participates in acid-base catalysis (mentioned in the enzyme chapter).</span></p><p><span> </span></p><h4 id='ⅲ-calculation-of-isoelectric-points'><span>(Ⅲ) Calculation of isoelectric points</span></h4><ul><li><p><span>The acidic amino acid, which is fully protonated, has a net charge of +1, so it can only be net charged to 0 if the α-COOH is completely dissociated and the R group is not dissociated</span></p><p><img src="media/3/酸性氨基酸pI.jpg" style="zoom: 25%;" /></p></li><li><p><span>Basic amino acids have a net charge of +2 when fully protonated, thus only when α-COOH and R group are fully dissociated, and α-NH</span><sup><span>3+</span></sup><span> remains undissociated, can the net charge be 0.</span></p><p><img src="media/3/碱性氨基酸pI.jpg" style="zoom: 25%;" /></p></li><li><p><span>Neutral amino acids have a net charge of 0 only when α-COOH is fully dissociated and α-NH</span><sup><span>3+</span></sup><span> remains undissociated.</span></p></li></ul><p><img src="media/3/中性氨基酸pI.jpg" style="zoom: 25%;" /></p><p><span> </span></p><h2 id='35chemical-reactions-of-amino-acids'><span>3.5.Chemical reactions of amino acids</span></h2><h3 id='351reactions-involving-α-nh2'><span>3.5.1.Reactions involving α-NH</span><sub><span>2</span></sub></h3><ul><li><p><strong><span>Sanger (DNFB) Method</span></strong></p><p><span>Reaction of 2,4-dinitrofluorobenzene with the N-terminal amino acid of the peptide chain forms DNP-peptide. Subsequent acid hydrolysis breaks the peptide bond, yielding numerous amino acids and a </span><mark><span>yellow DNP-Aa</span></mark><span>, which is then identified by chromatography to determine the type of amino acid.</span></p></li><li><p><strong><span>Edman (PITC) Method</span></strong></p><p><span>Reaction of phenyl isothiocyanate with the N-terminal amino acid of the peptide chain forms PTC-peptide. After acid hydrolysis, the peptide bond between the N-terminal PTC-Aa and the peptide is cleaved, resulting in a PTC-amino acid and a peptide with one fewer amino acid. The former is identified by chromatographic separation, while the latter enters the next round of the Edman cycle.</span></p></li><li><p><span>Free α-NH</span><sub><span>2</span></sub><span> reacts with aldehydes to produce </span><strong><span>Schiff bases</span></strong><span>.</span></p></li></ul><p>&nbsp;</p><h3 id='352reactions-involving-α-cooh'><span>3.5.2.Reactions involving α-COOH</span></h3><ul><li><strong><span>Esterification</span></strong><span>: Reaction of alcohol with acid to form an ester.</span></li><li><strong><span>Salt formation</span></strong><span>: Reaction of base with acid to form a salt.</span></li><li><strong><span>Acid chloride formation</span></strong><span>: -COOH reacts with PCl</span><sub><span>5</span></sub><span> (phosphorus pentachloride) or SOCl</span><sub><span>2</span></sub><span> (thionyl chloride) to form the corresponding acid chloride.</span></li></ul><p>&nbsp;</p><h3 id='353reactions-involving-α-nh2-and-α-cooh'><span>3.5.3.Reactions involving α-NH</span><sub><span>2</span></sub><span> and α-COOH</span></h3><ul><li><strong><span>Peptide bond formation</span></strong><span>: Dehydration condensation between the α-NH</span><sub><span>2</span></sub><span> and α-COOH of adjacent amino acids forms a peptide bond. </span><strong><span>Protonated -COOH and deprotonated NH</span><sub><span>2</span></sub><span> are required for peptide bond formation</span></strong><span>.</span></li><li><strong><span>Ninhydrin reaction</span></strong><span>: Ninhydrin and α-amino acids are heated together in a </span><strong><span>weakly acidic environment</span></strong><span>, promoting deamination and decarboxylation of the amino acid. The released ammonia reacts with one molecule of reduced ninhydrin to form a blue-purple substance with a maximum OD</span><sub><span>570</span></sub><span>, or directly reacts with (hydroxy) proline without deamination, forming a yellow substance with a maximum OD</span><sub><span>440</span></sub><span>. </span><strong><span>This is an important method for quantitative determination of amino acids</span></strong><span>.</span></li></ul><div class="md-diagram-panel md-fences-adv-panel"><svg id="mermaidChart1" width="100%" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" height="157.20001220703125" style="max-width: 963.765625px;" viewBox="0 0.0000019073486328125 963.765625 157.20001220703125"><style>#mermaidChart1 {font-family:sans-serif;font-size:16px;fill:#333;}#mermaidChart1 .error-icon{fill:#552222;}#mermaidChart1 .error-text{fill:#552222;stroke:#552222;}#mermaidChart1 .edge-thickness-normal{stroke-width:2px;}#mermaidChart1 .edge-thickness-thick{stroke-width:3.5px;}#mermaidChart1 .edge-pattern-solid{stroke-dasharray:0;}#mermaidChart1 .edge-pattern-dashed{stroke-dasharray:3;}#mermaidChart1 .edge-pattern-dotted{stroke-dasharray:2;}#mermaidChart1 .marker{fill:#333333;stroke:#333333;}#mermaidChart1 .marker.cross{stroke:#333333;}#mermaidChart1 svg{font-family:sans-serif;font-size:16px;}#mermaidChart1 .label{font-family:sans-serif;color:#333;}#mermaidChart1 .cluster-label text{fill:#333;}#mermaidChart1 .cluster-label span{color:#333;}#mermaidChart1 .label text,#mermaidChart1 span{fill:#333;color:#333;}#mermaidChart1 .node rect,#mermaidChart1 .node circle,#mermaidChart1 .node ellipse,#mermaidChart1 .node polygon,#mermaidChart1 .node path{fill:#ECECFF;stroke:#9370DB;stroke-width:1px;}#mermaidChart1 .node .label{text-align:center;}#mermaidChart1 .node.clickable{cursor:pointer;}#mermaidChart1 .arrowheadPath{fill:#333333;}#mermaidChart1 .edgePath .path{stroke:#333333;stroke-width:2.0px;}#mermaidChart1 .flowchart-link{stroke:#333333;fill:none;}#mermaidChart1 .edgeLabel{background-color:#e8e8e8;text-align:center;}#mermaidChart1 .edgeLabel rect{opacity:0.5;background-color:#e8e8e8;fill:#e8e8e8;}#mermaidChart1 .cluster rect{fill:#ffffde;stroke:#aaaa33;stroke-width:1px;}#mermaidChart1 .cluster text{fill:#333;}#mermaidChart1 .cluster span{color:#333;}#mermaidChart1 div.mermaidTooltip{position:absolute;text-align:center;max-width:200px;padding:2px;font-family:sans-serif;font-size:12px;background:hsl(80, 100%, 96.2745098039%);border:1px solid #aaaa33;border-radius:2px;pointer-events:none;z-index:100;}#mermaidChart1 :root{--mermaid-font-family:sans-serif;}#mermaidChart1 :root{--mermaid-alt-font-family:sans-serif;}</style><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath LS-Aa+Anthraquinone LE-NH3+CO2+Reduced_Anthraquinone" id="L-Aa+Anthraquinone-NH3+CO2+Reduced_Anthraquinone" style="opacity: 1;"><path class="path" d="M282.8656311035156,30.799999237060547L523.0343856811523,30.799999237060547L655.5156402587891,30.799999237060547" marker-end="url(#arrowhead34)" style="fill:none;stroke-width:2px;stroke-dasharray:3;"></path><defs><marker id="arrowhead34" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-NH3+Reduced_Anthraquinone+Anthraquinone LE-Blue_substance" id="L-NH3+Reduced_Anthraquinone+Anthraquinone-Blue_substance" style="opacity: 1;"><path class="path" d="M390.5531311035156,126.39999771118164L523.0343856811523,126.39999771118164L737.1093902587891,126.39999771118164" marker-end="url(#arrowhead35)" style="fill:none"></path><defs><marker id="arrowhead35" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="translate(523.0343856811523,30.799999237060547)" style="opacity: 1;"><g transform="translate(-107.48125457763672,-12.800000190734863)" class="label"><rect rx="0" ry="0" width="214.96250915527344" height="25.600000381469727"></rect><foreignObject width="214.96250915527344" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Aa+Anthraquinone-NH3+CO2+Reduced_Anthraquinone" class="edgeLabel L-LS-Aa+Anthraquinone' L-LE-NH3+CO2+Reduced_Anthraquinone">weak_acidic_conditions,heat</span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-NH3+Reduced_Anthraquinone+Anthraquinone-Blue_substance" class="edgeLabel L-LS-NH3+Reduced_Anthraquinone+Anthraquinone' L-LE-Blue_substance"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node default" id="flowchart-Aa+Anthraquinone-28" transform="translate(199.2765655517578,30.799999237060547)" style="opacity: 1;"><rect rx="0" ry="0" x="-83.58906555175781" y="-22.800000190734863" width="167.17813110351562" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-73.58906555175781,-12.800000190734863)"><foreignObject width="147.17813110351562" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Aa+Anthraquinone</div></foreignObject></g></g></g><g class="node default" id="flowchart-NH3+CO2+Reduced_Anthraquinone-29" transform="translate(805.6406402587891,30.799999237060547)" style="opacity: 1;"><rect rx="0" ry="0" x="-150.125" y="-22.800000190734863" width="300.25" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-140.125,-12.800000190734863)"><foreignObject width="280.25" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">NH3+CO2+Reduced_Anthraquinone</div></foreignObject></g></g></g><g class="node default" id="flowchart-NH3+Reduced_Anthraquinone+Anthraquinone-30" transform="translate(199.2765655517578,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-191.2765655517578" y="-22.800000190734863" width="382.5531311035156" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-181.2765655517578,-12.800000190734863)"><foreignObject width="362.5531311035156" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">NH3+Reduced_Anthraquinone+Anthraquinone</div></foreignObject></g></g></g><g class="node default" id="flowchart-Blue_substance-31" transform="translate(805.6406402587891,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-68.53125" y="-22.800000190734863" width="137.0625" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58.53125,-12.800000190734863)"><foreignObject width="117.0625" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Blue_substance</div></foreignObject></g></g></g></g></g></g></svg></div><p>&nbsp;</p><h3 id='354reactions-involving-r-groups'><span>3.5.4.Reactions involving R groups</span></h3><p><strong><span>1. Phenyl group</span></strong><span>: Reacts with concentrated nitric acid to produce a yellow substance.</span></p><p><strong><span>2. Thiol group (Cys)</span></strong></p><p><span>For thiol enzymes, exposure to alkylating agents or heavy metal ions that can disrupt the thiol -SH group leads to enzyme inactivation. Therefore, reducing agents such as GSH act as enzyme protectants or activators.</span></p><ul><li><p><span>Transition from -S-S- to -SH:</span></p><ul><li><strong><span>Reduction</span></strong><span>: β-mercaptoethanol, dithiothreitol, reduced glutathione (GSH)</span></li><li><strong><span>Oxidation</span></strong><span>: Formic acid</span></li></ul></li><li><p><span>Destruction of thiol groups:</span></p><ul><li><span>Alkylation: Iodoacetic acid, iodoacetamide, p-chloromercuribenzoic acid</span></li><li><span>Reaction with heavy metal ions</span></li><li><span>Reaction with organic mercury, organic arsenic</span></li></ul></li></ul><p><strong><span>3. Guanidino group (Arg)</span></strong><span>: Reacts with α-naphthol and hypobromite to produce a red substance (the Sakaguchi reaction).</span></p><p><strong><span>4. Phenolic hydroxyl group (Tyr)</span></strong></p><ul><li><span>Reacts with diazonium compounds to synthesize an orange substance (the Pauly reaction).</span></li><li><span>Reacts with nitric acid, mercuric nitrate, etc., to synthesize a red substance (the Millon&#39;s reaction).</span></li><li><span>Reacts with the molybdenum blue or tungsten blue in Folin-phenol reagent to synthesize blue phosphomolybdate or phosphotungstate salts.</span></li></ul><p><strong><span>5. Indole group (Trp)</span></strong></p><ul><li><span>Reacts with acetic aldehyde, concentrated sulfuric acid to produce a red substance (the acetic aldehyde reaction).</span></li><li><span>Reacts with the molybdenum blue or tungsten blue in Folin-phenol reagent to synthesize blue phosphomolybdate or phosphotungstate salts.</span></li></ul><p><strong><span>6. Imidazole group</span></strong></p><ul><li><span>Reacts with diazonium compounds to synthesize a red-brown substance (the Pauly reaction).</span></li></ul><h2 id='36peptides-and-peptide-bonds'><span>3.6.Peptides and Peptide Bonds</span></h2><p><span> </span><img src="media/3/肽键形成.jpg" style="zoom: 33%;" /></p><p><span>After the polymerization of amino acids, the resulting polymer is called a polypeptide. A single polypeptide or several interacting polypeptides can form a protein.</span></p><p><strong><span>Peptide bond</span></strong><span>	</span><span>In a polypeptide chain, it refers to the covalent bond formed by the dehydration condensation between the α-COOH of one amino acid and the α-NH</span><sub><span>2</span></sub><span> of its adjacent amino acid. </span></p><p><span>Characteristics of the peptide bond:</span></p><ol start='' ><li><span>The C-N bond exhibits properties intermediate between a double bond and a single bond, </span><strong><span>having partial double bond characteristics</span></strong><span>. It is not freely rotatable, shorter than a single bond, and longer than a double bond. Therefore, only the N-H, C</span><sub><span>α</span></sub><span>-N, and C</span><sub><span>α</span></sub><span>-C bonds in a peptide plane can rotate.</span></li><li><span>The majority of peptide bonds are in the trans configuration (as shown in the figure), while only the peptide bonds formed between Proline and other amino acids have a small probability (1/10) of being in the </span><strong><span>cis configuration</span></strong><span>.</span></li><li><span>Peptide bonds are catalyzed by 23s rRNA (in prokaryotes) during the translation process.</span></li></ol><p>&nbsp;</p><p><strong><span>Peptide plane</span></strong><span>	</span><span>It refers to the </span><strong><span>rigid plane</span></strong><span> formed by the four atoms that make up the peptide bond and the two adjacent C</span><sub><span>α</span></sub><span> atoms. This plane cannot rotate internally, but rotation is possible between planes, leading to different configurations of the peptide chain.</span></p><p><span>After dehydration condensation of amino acids, they are referred to as amino acid residues. The number of amino acid residues (peptide bond count + 1) determines the classification as a peptide. A chain with fewer than 20 residues is generally referred to as an </span><strong><span>oligopeptide</span></strong><span>, while those with more than 20 residues are called </span><strong><span>polypeptides</span></strong><span>.</span></p><p><img src="media/3/肽平面.png" style="zoom: 73%;" /></p><p>&nbsp;</p><p><strong><span>Peptide unit</span></strong><span>	</span><span>refers to the basic unit of a peptide chain, which consists of the four atoms involved in the peptide bond and the two adjacent C</span><sub><span>α</span></sub><span> atoms. Two peptide units (one peptide bond) form a dipeptide, and so on. Therefore, the peptide unit can be considered equivalent to the peptide plane. As shown in the figure, a clear carbon skeleton can be observed: C</span><sub><span>α</span></sub><span>-C-N-C</span><sub><span>α</span></sub><span>-C-N-C</span><sub><span>α</span></sub><span>-C...</span></p><p><img src="media/3/多肽链.jpg" style="zoom: 33%;" /></p><p>&nbsp;</p><p><span>The two ends of a polypeptide chain are the free -COOH (carboxyl) group end and the free -NH</span><sub><span>2</span></sub><span> (amino) group end:</span></p><ul><li><strong><span>Writing and reading direction of a peptide chain: N-terminal -&gt; C-terminal</span></strong></li><li><span>Synthesis (translation) direction of a peptide chain in the body: N-terminal -&gt; C-terminal</span></li><li><span>Direction of artificial synthesis of a peptide chain outside the body: C-terminal -&gt; N-terminal</span></li></ul><p>&nbsp;</p><h2 id='37other-molecular-forces'><span>3.7.Other Molecular Forces</span></h2><p><span>The peptide bond mentioned earlier is a </span><strong><span>covalent bond</span></strong><span>, and in addition to that, there are </span><strong><span>four types of non-covalent bonds and disulfide bonds</span></strong><span> present in proteins.</span></p><p><img src="media/3/非共价作用力.jpg" style="zoom: 73%;" /></p><p><strong><span>(1) Ionic bond</span></strong><span>: Refers to the electrostatic attraction between groups carrying opposite charges, and it is a </span><strong><span>non-covalent bond</span></strong><span>.</span></p><p><strong><span>(2) Hydrogen bond</span></strong><span>: Refers to the mutual attraction between highly electronegative N or O atoms and H-N or H-O bonds.</span></p><ul><li><span>It is a </span><strong><span>non-covalent bond</span></strong><span> that stabilizes the </span><strong><span>secondary structure</span></strong><span>.</span></li><li><span>It is directional and saturable.</span></li><li><span>Disruption methods: urea, guanidine hydrochloride.</span></li></ul><p><strong><span>(3) Hydrophobic force</span></strong><span>: Refers to the force where hydrophobic groups avoid the aqueous environment and tend to aggregate.</span></p><ul><li><span>It is the most important </span><strong><span>non-covalent force</span></strong><span> in maintaining the stability of the </span><strong><span>tertiary and quaternary structures</span></strong><span>.</span></li><li><span>Disruption method: SDS.</span></li></ul><p><strong><span>(4) Disulfide bond</span></strong><span>: Refers to the </span><strong><span>covalent bond</span></strong><span> formed between two different cysteine residues within the same or different peptide chains, which contributes to the stability of protein structure.</span></p><ul><li><p><span>Disruption of disulfide bonds can occur through:</span></p><ul><li><span>Reduction: beta-mercaptoethanol, dithiothreitol, reduced form of glutathione (GSH).</span></li><li><span>Oxidation: performic acid.</span></li></ul></li></ul><p><strong><span>(unlabeled) Van der Waals forces</span></strong><span>: The weakest intermolecular force among the four non-covalent forces.</span></p><p>&nbsp;</p><p>&nbsp;</p><h2 id='38active-peptides-and-polypeptides'><strong><span>3.8.Active peptides and Polypeptides</span></strong></h2><p><strong><span>Active peptides</span></strong><span>	</span><span>Refers to oligopeptides and polypeptides that have biological activity and can regulate physiological metabolic activities.</span></p><ul><li><strong><span>GSH (Glutathione)</span></strong><span>: A tripeptide formed by the condensation of Glu, Cys, and Gly, with a hydrogen bond between the γ-COOH of Glu and the α-NH</span><sub><span>2</span></sub><span> of Cys. It is a natural antioxidant in the body due to the presence of a free and easily oxidizable -SH group. It protects the reduction of sulfhydryl groups and the reduction of hemoglobin Fe</span><sup><span>2+</span></sup><span>.</span></li><li><span>Oxytocin: A nonapeptide that promotes uterine contractions and milk secretion.</span></li><li><span>Vasopressin: A nonapeptide that increases blood pressure (Oxytocin and vasopressin are both nonapeptides, but they differ in the amino acids at positions 3 and 8, resulting in distinct functions).</span></li><li><span>Enkephalin: Produced in the nervous system, it is used for analgesia.</span></li><li><span>Thaumatin: A dipeptide, it is an artificial sweetener.</span></li><li><span>Streptomycin: Essentially a peptide antibiotic, it inhibits transcription by binding to DNA.</span></li></ul><p>&nbsp;</p><p><span>Most of the active peptides introduced in the previous section are oligopeptides or small polypeptides, and their lengths generally do not exceed 50 amino acid residues. How long can the polypeptide chain of a protein be? The lengths of polypeptide chains in different proteins vary: for example, insulin contains an A chain with 30 residues and a B chain with 21 residues, and these two chains are connected by disulfide bonds (not non-covalent bonds, so insulin is a monomeric protein). In 1965, Chinese scientists first artificially synthesized crystalline bovine insulin. On the other hand, zombie cytochrome C (Cyt C) contains a polypeptide chain with 104 residues, ribonuclease A (RNase A) contains a polypeptide chain with 124 residues, and zombie hemoglobin (Hb) contains four subunits, totaling 574 residues. These examples illustrate the diversity in the size and structure of proteins. Detailed explanations of protein structures will be provided in </span><strong><span>Section 2: Protein Structure and Function</span></strong><span>.</span></p><p><span>Due to the different physiological functions and properties of amino acids in proteins, the composition of the 20 amino acids varies in different proteins. For example, hemoglobin contains a large amount of His in its primary structure, and a certain amino acid may only appear once in a particular protein. On average, the relative molecular mass (Mr) of the 20 amino acids is 138. However, due to the dehydration condensation required for peptide bond formation, the average relative molecular mass of amino acids in proteins is approximately </span><strong><span>128-18=110 Da.</span></strong></p><p>&nbsp;</p><h2 id='39basic-characteristics-of-proteins'><strong><span>3.9.Basic Characteristics of Proteins</span></strong></h2><p><span>Elemental Composition: Proteins are composed of C, O, N, H, and small amounts of S, with S being a characteristic element (distinguishing it from nucleic acids, which contain P). </span><strong><span>Proteins share the common feature of containing an average of 16% nitrogen</span></strong><span>, which forms the basis for calculating nitrogen content using the Kjeldahl method.</span></p><ul><li><p><span>Protein Coefficient: Since proteins contain an average of 16% nitrogen, 1g of nitrogen represents 6.25g of protein. This coefficient is referred to as the protein coefficient.</span></p><blockquote><p><span>Nitrogen Content × 6.25 = Protein Content</span></p></blockquote></li></ul><p>&nbsp;</p><p><strong><span>Colloidal Nature</span></strong><span>: Proteins are colloidal molecules with sizes ranging from 1-100nm, falling within the range of colloidal particle sizes. As a result, proteins exhibit the Tyndall effect, Brownian motion, </span><strong><span>and cannot pass through semi-permeable membranes.</span></strong><span> The stability of protein solutions as stable hydrophilic colloids is attributed to two factors:</span></p><ul><li><strong><span>Hydration Layer</span></strong><span>: The surface of protein molecules contains a large number of hydrophilic groups, which can adsorb water molecules to form a hydration layer. This prevents protein molecules from colliding and aggregating during movement, thereby increasing the solubility of protein solutions and enhancing stability.</span></li><li><strong><span>Double Layer</span></strong><span>: The surface of protein molecules contains a large number of dissociable groups. In a solution at a specific pH, protein molecules will dissociate into ions with the same charge, attracting counterions in the solution to form a double layer. Based on the principle of repulsion between like charges, this prevents proteins from colliding and aggregating, thereby increasing the solubility and stability of protein solutions.</span></li></ul><p><span>Therefore, when the hydration layer and double layer are disrupted, proteins are more likely to precipitate.</span></p><p>&nbsp;</p><p><strong><span>Amphiprotic Nature</span></strong><span>: Polypeptide chains contain ionizable α-NH</span><sub><span>2</span></sub><span>, α-COOH, and R groups. For short peptides, the electrostatic properties of the molecule depend mainly on α-NH</span><sub><span>2</span></sub><span>, α-COOH, and R groups. However, for large polypeptide or protein molecules, because α-NH</span><sub><span>2</span></sub><span> and α-COOH are far apart and have weak electrostatic attraction, the pKa of -COOH is slightly larger than that of free amino acids, and the pKa of -NH</span><sub><span>2</span></sub><span> is slightly smaller, so the dissociated groups are mainly R groups. Therefore, in solutions at different pH levels, protein molecules can undergo varying degrees of dissociation, resulting in a net positive charge, net negative charge, or becoming electrically neutral in the form of amphoteric ions.</span></p><ul><li><span>Proteins with a higher content of basic amino acids have a </span><strong><span>pI &gt; 7</span></strong><span>, while those with a higher content of acidic amino acids have a </span><strong><span>pI &lt; 7</span></strong><span>.</span></li><li><span>Factors affecting the isoelectric point of a protein: amino acid composition, ionic strength of the solution, temperature, etc.</span></li><li><span>Plasma dots: The pH at which a protein is uncharged in an ion-free solution. </span><strong><span>The plasma dots is a characteristic constant of a protein, rather than pI.</span></strong></li></ul><p>&nbsp;</p><p><strong><span>UV Absorption</span></strong><span>: Almost all proteins contain aromatic amino acids, resulting in maximum absorption at OD</span><sub><span>280</span></sub><span>, and the absorbance within a certain range is directly proportional to the protein content. Therefore, UV absorption is one of the methods used to measure protein content.</span></p><p>&nbsp;</p><p>&nbsp;</p><h2 id='310composition-and-classification-of-proteins'><strong><span>3.10.Composition and Classification of Proteins</span></strong></h2><div class="md-diagram-panel md-fences-adv-panel"><svg id="mermaidChart2" width="100%" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" height="252.79998779296875" style="max-width: 1461.9000244140625px;" viewBox="0 0.0000019073486328125 1461.9000244140625 252.79998779296875"><style>#mermaidChart2 {font-family:sans-serif;font-size:16px;fill:#333;}#mermaidChart2 .error-icon{fill:#552222;}#mermaidChart2 .error-text{fill:#552222;stroke:#552222;}#mermaidChart2 .edge-thickness-normal{stroke-width:2px;}#mermaidChart2 .edge-thickness-thick{stroke-width:3.5px;}#mermaidChart2 .edge-pattern-solid{stroke-dasharray:0;}#mermaidChart2 .edge-pattern-dashed{stroke-dasharray:3;}#mermaidChart2 .edge-pattern-dotted{stroke-dasharray:2;}#mermaidChart2 .marker{fill:#333333;stroke:#333333;}#mermaidChart2 .marker.cross{stroke:#333333;}#mermaidChart2 svg{font-family:sans-serif;font-size:16px;}#mermaidChart2 .label{font-family:sans-serif;color:#333;}#mermaidChart2 .cluster-label text{fill:#333;}#mermaidChart2 .cluster-label span{color:#333;}#mermaidChart2 .label text,#mermaidChart2 span{fill:#333;color:#333;}#mermaidChart2 .node rect,#mermaidChart2 .node circle,#mermaidChart2 .node ellipse,#mermaidChart2 .node polygon,#mermaidChart2 .node path{fill:#ECECFF;stroke:#9370DB;stroke-width:1px;}#mermaidChart2 .node .label{text-align:center;}#mermaidChart2 .node.clickable{cursor:pointer;}#mermaidChart2 .arrowheadPath{fill:#333333;}#mermaidChart2 .edgePath .path{stroke:#333333;stroke-width:2.0px;}#mermaidChart2 .flowchart-link{stroke:#333333;fill:none;}#mermaidChart2 .edgeLabel{background-color:#e8e8e8;text-align:center;}#mermaidChart2 .edgeLabel rect{opacity:0.5;background-color:#e8e8e8;fill:#e8e8e8;}#mermaidChart2 .cluster rect{fill:#ffffde;stroke:#aaaa33;stroke-width:1px;}#mermaidChart2 .cluster text{fill:#333;}#mermaidChart2 .cluster span{color:#333;}#mermaidChart2 div.mermaidTooltip{position:absolute;text-align:center;max-width:200px;padding:2px;font-family:sans-serif;font-size:12px;background:hsl(80, 100%, 96.2745098039%);border:1px solid #aaaa33;border-radius:2px;pointer-events:none;z-index:100;}#mermaidChart2 :root{--mermaid-font-family:sans-serif;}#mermaidChart2 :root{--mermaid-alt-font-family:sans-serif;}</style><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath LS-Classification LE-shape" id="L-Classification-shape" style="opacity: 1;"><path class="path" d="M747.5633010864258,36.28168544261605L285.12657165527344,78.5999984741211L285.12657165527344,103.5999984741211" marker-end="url(#arrowhead55)" style="fill:none"></path><defs><marker id="arrowhead55" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-shape LE-Globular-proteins" id="L-shape-Globular-proteins" style="opacity: 1;"><path class="path" d="M252.42657089233398,134.21352727754117L85.08125305175781,174.1999969482422L85.08125305175781,199.1999969482422" marker-end="url(#arrowhead56)" style="fill:none"></path><defs><marker id="arrowhead56" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-shape LE-Fibrous-proteins" id="L-shape-Fibrous-proteins" style="opacity: 1;"><path class="path" d="M285.12657165527344,149.1999969482422L285.12657165527344,174.1999969482422L285.12657165527344,199.1999969482422" marker-end="url(#arrowhead57)" style="fill:none"></path><defs><marker id="arrowhead57" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-shape LE-Membrane-proteins" id="L-shape-Membrane-proteins" style="opacity: 1;"><path class="path" d="M317.8265724182129,133.85439976796135L494.80938720703125,174.1999969482422L494.80938720703125,199.1999969482422" marker-end="url(#arrowhead58)" style="fill:none"></path><defs><marker id="arrowhead58" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Classification LE-composition" id="L-Classification-composition" style="opacity: 1;"><path class="path" d="M807.4648628234863,53.599998474121094L807.4648628234863,78.5999984741211L807.4648628234863,103.5999984741211" marker-end="url(#arrowhead59)" style="fill:none"></path><defs><marker id="arrowhead59" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-composition LE-Simple-proteins" id="L-composition-Simple-proteins" style="opacity: 1;"><path class="path" d="M757.3345990674549,149.1999969482422L702.3672027587891,174.1999969482422L702.3672027587891,199.1999969482422" marker-end="url(#arrowhead60)" style="fill:none"></path><defs><marker id="arrowhead60" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-composition LE-Conjugated-proteins" id="L-composition-Conjugated-proteins" style="opacity: 1;"><path class="path" d="M857.5951265795178,149.1999969482422L912.5625228881836,174.1999969482422L912.5625228881836,199.1999969482422" marker-end="url(#arrowhead61)" style="fill:none"></path><defs><marker id="arrowhead61" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-Classification LE-structure" id="L-Classification-structure" style="opacity: 1;"><path class="path" d="M867.3664245605469,37.2139986242122L1253.878158569336,78.5999984741211L1253.878158569336,103.5999984741211" marker-end="url(#arrowhead62)" style="fill:none"></path><defs><marker id="arrowhead62" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-structure LE-Monomeric-proteins" id="L-structure-Monomeric-proteins" style="opacity: 1;"><path class="path" d="M1209.8031578063965,145.04488570989633L1140.8828430175781,174.1999969482422L1140.8828430175781,199.1999969482422" marker-end="url(#arrowhead63)" style="fill:none"></path><defs><marker id="arrowhead63" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g><g class="edgePath LS-structure LE-Oligomeric-proteins" id="L-structure-Oligomeric-proteins" style="opacity: 1;"><path class="path" d="M1297.9531593322754,145.04488570989633L1366.8734741210938,174.1999969482422L1366.8734741210938,199.1999969482422" marker-end="url(#arrowhead64)" style="fill:none"></path><defs><marker id="arrowhead64" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" class="arrowheadPath" style="stroke-width: 1; stroke-dasharray: 1, 0;"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Classification-shape" class="edgeLabel L-LS-Classification' L-LE-shape"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-shape-Globular-proteins" class="edgeLabel L-LS-shape' L-LE-Globular-proteins"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-shape-Fibrous-proteins" class="edgeLabel L-LS-shape' L-LE-Fibrous-proteins"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-shape-Membrane-proteins" class="edgeLabel L-LS-shape' L-LE-Membrane-proteins"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Classification-composition" class="edgeLabel L-LS-Classification' L-LE-composition"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-composition-Simple-proteins" class="edgeLabel L-LS-composition' L-LE-Simple-proteins"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-composition-Conjugated-proteins" class="edgeLabel L-LS-composition' L-LE-Conjugated-proteins"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-Classification-structure" class="edgeLabel L-LS-Classification' L-LE-structure"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-structure-Monomeric-proteins" class="edgeLabel L-LS-structure' L-LE-Monomeric-proteins"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><rect rx="0" ry="0" width="0" height="0"></rect><foreignObject width="0" height="0"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;"><span id="L-L-structure-Oligomeric-proteins" class="edgeLabel L-LS-structure' L-LE-Oligomeric-proteins"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node default" id="flowchart-Classification-49" transform="translate(807.4648628234863,30.799999237060547)" style="opacity: 1;"><rect rx="0" ry="0" x="-59.90156173706055" y="-22.800000190734863" width="119.8031234741211" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-49.90156173706055,-12.800000190734863)"><foreignObject width="99.8031234741211" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Classification</div></foreignObject></g></g></g><g class="node default" id="flowchart-shape-50" transform="translate(285.12657165527344,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-32.70000076293945" y="-22.800000190734863" width="65.4000015258789" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-22.700000762939453,-12.800000190734863)"><foreignObject width="45.400001525878906" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">shape</div></foreignObject></g></g></g><g class="node default" id="flowchart-Globular-proteins-51" transform="translate(85.08125305175781,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-77.08125305175781" y="-22.800000190734863" width="154.16250610351562" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-67.08125305175781,-12.800000190734863)"><foreignObject width="134.16250610351562" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Globular-proteins</div></foreignObject></g></g></g><g class="node default" id="flowchart-Fibrous-proteins-53" transform="translate(285.12657165527344,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-72.96406173706055" y="-22.800000190734863" width="145.9281234741211" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-62.96406173706055,-12.800000190734863)"><foreignObject width="125.9281234741211" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Fibrous-proteins</div></foreignObject></g></g></g><g class="node default" id="flowchart-Membrane-proteins-55" transform="translate(494.80938720703125,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-86.71875" y="-22.800000190734863" width="173.4375" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-76.71875,-12.800000190734863)"><foreignObject width="153.4375" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Membrane-proteins</div></foreignObject></g></g></g><g class="node default" id="flowchart-composition-57" transform="translate(807.4648628234863,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-57.75" y="-22.800000190734863" width="115.5" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-47.75,-12.800000190734863)"><foreignObject width="95.5" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">composition</div></foreignObject></g></g></g><g class="node default" id="flowchart-Simple-proteins-58" transform="translate(702.3672027587891,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-70.83906173706055" y="-22.800000190734863" width="141.6781234741211" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-60.83906173706055,-12.800000190734863)"><foreignObject width="121.6781234741211" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Simple-proteins</div></foreignObject></g></g></g><g class="node default" id="flowchart-Conjugated-proteins-60" transform="translate(912.5625228881836,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-89.35625457763672" y="-22.800000190734863" width="178.71250915527344" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-79.35625457763672,-12.800000190734863)"><foreignObject width="158.71250915527344" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Conjugated-proteins</div></foreignObject></g></g></g><g class="node default" id="flowchart-structure-62" transform="translate(1253.878158569336,126.39999771118164)" style="opacity: 1;"><rect rx="0" ry="0" x="-44.07500076293945" y="-22.800000190734863" width="88.1500015258789" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-34.07500076293945,-12.800000190734863)"><foreignObject width="68.1500015258789" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">structure</div></foreignObject></g></g></g><g class="node default" id="flowchart-Monomeric-proteins-63" transform="translate(1140.8828430175781,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-88.96406555175781" y="-22.800000190734863" width="177.92813110351562" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-78.96406555175781,-12.800000190734863)"><foreignObject width="157.92813110351562" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Monomeric-proteins</div></foreignObject></g></g></g><g class="node default" id="flowchart-Oligomeric-proteins-65" transform="translate(1366.8734741210938,221.99999618530273)" style="opacity: 1;"><rect rx="0" ry="0" x="-87.02656555175781" y="-22.800000190734863" width="174.05313110351562" height="45.60000038146973" class="label-container"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-77.02656555175781,-12.800000190734863)"><foreignObject width="154.05313110351562" height="25.600000381469727"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; white-space: nowrap;">Oligomeric-proteins</div></foreignObject></g></g></g></g></g></g></svg></div><p>&nbsp;</p><h3 id='3101classification-based-on-shape'><span>3.10.1.Classification based on shape</span></h3><p><strong><span>Globular proteins</span></strong><span>	</span><span>refers to a class of proteins that are approximately spherical or elliptical in shape.</span></p><ul><li><span>Hydrophilic amino acids are mostly exposed on the outer surface, while hydrophobic amino acids are mostly buried inside the molecule, making them highly water-soluble.</span></li><li><span>They are composed of various secondary structures and have a clearer hierarchy of structure.</span></li><li><span>Most enzymes and functional proteins in cells belong to globular proteins.</span></li></ul><p><strong><span>Fibrous proteins</span></strong><span>	</span><span>refers to a class of proteins that are approximately elongated or rod-shaped in structure.</span></p><ul><li><span>They are less water-soluble compared to globular proteins.</span></li><li><span>They often function as structural proteins, providing support and protection in organisms. Examples include alpha-keratin, fibroin, elastin, and collagen.</span></li><li><span>They are mostly composed of a single type of secondary structure, such as alpha-helices in alpha-keratin and beta-sheets in fibroin.</span></li></ul><p><strong><span>Membrane proteins</span></strong><span>	</span><span>refer tp proteins that are associated with various membrane systems (plasma membrane + internal membranes) of cells, including peripheral proteins, integral membrane proteins, and anchored proteins.</span></p><p>&nbsp;</p><h3 id='3102classification-based-on-composition'><span>3.10.2.Classification based on composition</span></h3><p><strong><span>Simple proteins</span></strong><span>: Refers to proteins that are composed solely of amino acids.</span></p><ul><li><span>Examples include globulins, albumins (transport fatty acids), histones, protamines, glutenins, scleroproteins, lysozyme, ribonuclease A (RNase A), etc.</span></li></ul><p><strong><span>Conjugated proteins</span></strong><span>: Refers to proteins that, in addition to amino acids, also contain prosthetic groups or ligands. The portion of the protein without the prosthetic group or ligand is called the apoprotein.</span></p><figure><table><thead><tr><th><span>Type</span></th><th><span>Examples</span></th></tr></thead><tbody><tr><td><span>Glycoproteins</span></td><td><span>Collagen, membrane proteins, transferrin, immunoglobulins, interferons, thrombin</span></td></tr><tr><td><span>Lipoproteins</span></td><td><span>CM, VLDL (very low-density lipoprotein), LDL (low-density lipoprotein), HDL (high-density lipoprotein)</span></td></tr><tr><td><span>Phosphoproteins</span></td><td><span>Glycogen phosphorylase, casein</span></td></tr><tr><td><span>Nucleoproteins</span></td><td><span>DNP (DNA + protein), RNP (RNA + protein)</span></td></tr><tr><td><span>Hemoproteins</span></td><td><span>Hemoglobin, myoglobin, cytochromes, catalase</span></td></tr><tr><td><span>Flavoproteins</span></td><td><span>Succinate dehydrogenase (FADH</span><sub><span>2</span></sub><span>), dihydrolipoyl dehydrogenase (FADH</span><sub><span>2</span></sub><span>), acyl-CoA dehydrogenase (FADH</span><sub><span>2</span></sub><span>), NADH dehydrogenase (NADH)</span></td></tr><tr><td><span>Metalloproteins</span></td><td><span>Alcohol dehydrogenase, carboxypeptidase (Zn</span><sup><span>2+</span></sup><span>), pyruvate carboxylase (Mn</span><sup><span>2+</span></sup><span>), alpha-amylase (Ca</span><sup><span>2+</span></sup><span>), nitrogenase (Mo, Fe)</span></td></tr></tbody></table></figure><p>&nbsp;</p><h3 id='3103classification-based-on-structure'><span>3.10.3.Classification based on structure</span></h3><p><strong><span>Monomeric proteins</span></strong><span>: Refers to proteins that consist of a single polypeptide chain or subunit.</span></p><ul><li><span>Examples include lysozyme, myoglobin (Mb), ribonuclease A (RNase A), insulin, etc.</span></li></ul><p><strong><span>Oligomeric proteins</span></strong><span>: Refers to proteins that contain multiple subunits, which are held together by non-covalent bonds.</span></p><blockquote><ol start='' ><li><span>There are four types of non-covalent bonds: hydrophobic interactions, hydrogen bonds, ionic bonds, and van der Waals forces. Although insulin has two peptide chains, they are connected by disulfide bonds (covalent bonds). Therefore, the two peptide chains can be considered as one subunit, making insulin a monomeric protein.</span></li><li><span>The molecular weight of an amino acid residue in a protein is 110 Da, so it can be calculated by dividing the molecular weight of the protein by 110 to obtain the number of amino acid residues. This calculation only applies to monomeric proteins.</span></li></ol></blockquote><ul><li><span>Examples include hemoglobin and most functional proteins and enzymes.</span></li></ul><p>&nbsp;</p><p>&nbsp;</p><div class='footnotes-area'  ><hr/>
<div class='footnote-line'><span class='md-fn-count'>1</span> <span>For example, selenocysteine, pyrrole tyrosine, hydroxyproline, hydroxylysine, etc </span> <a name='dfref-footnote-1' href='#ref-footnote-1' title='回到文档' class='reversefootnote' >↩</a></div>
<div class='footnote-line'><span class='md-fn-count'>2</span> <span>or example, β-alanine, γ-aminobutyric acid, ornithine, citrulline, etc </span> <a name='dfref-footnote-2' href='#ref-footnote-2' title='回到文档' class='reversefootnote' >↩</a></div>
<div class='footnote-line'><span class='md-fn-count'>3</span> <span>Chiral carbon: Four different groups are attached to a carbon atom</span> <a name='dfref-footnote-3' href='#ref-footnote-3' title='回到文档' class='reversefootnote' >↩</a></div></div></div></div>
</div>


</div>
</main>

<script src="bootstrap/assets/dist/js/bootstrap.bundle.min.js"></script>
<script>(function(){var e=document.body.parentElement,t=[],n=null,i=document.body.classList.contains("typora-export-collapse-outline"),r=function(e,t,n){document.addEventListener(e,function(e){if(!e.defaultPrevented)for(var i=e.target;i&&i!=this;i=i.parentNode)if(i.matches(t)){!1===n.call(i,e)&&(e.preventDefault(),e.stopPropagation());break}},!1)};function o(){return e.scrollTop}r("click",".outline-expander",function(e){var t=this.closest(".outline-item-wrapper").classList;return t.contains("outline-item-open")?t.remove("outline-item-open"):t.add("outline-item-open"),d(),!1}),r("click",".outline-item",function(e){var t=this.querySelector(".outline-label");if(location.hash="#"+t.getAttribute("href"),i){var n=this.closest(".outline-item-wrapper").classList;n.contains("outline-item-open")||n.add("outline-item-open"),c(),n.add("outline-item-active")}});var a,s,l=function(){var e=o();n=null;for(var i=0;i<t.length&&t[i][1]-e<60;i++)n=t[i]},c=function(){document.querySelectorAll(".outline-item-active").forEach(e=>e.classList.remove("outline-item-active")),document.querySelectorAll(".outline-item-single.outline-item-open").forEach(e=>e.classList.remove("outline-item-open"))},d=function(){if(n){c();var e=document.querySelector('.outline-label[href="#'+(CSS.escape?CSS.escape(n[0]):n[0])+'"]');if(e)if(i){var t=e.closest(".outline-item-open>ul>.outline-item-wrapper");if(t)t.classList.add("outline-item-active");else{for(var r=(e=e.closest(".outline-item-wrapper")).parentElement.closest(".outline-item-wrapper");r;)r=(e=r).parentElement.closest(".outline-item-wrapper");e.classList.add("outline-item-active")}}else e.closest(".outline-item-wrapper").classList.add("outline-item-active")}};window.addEventListener("scroll",function(e){a&&clearTimeout(a),a=setTimeout(function(){l(),d()},300)});var u=function(){s=setTimeout(function(){!function(){t=[];var e=o();document.querySelector("#write").querySelectorAll("h1, h2, h3, h4, h5, h6").forEach(n=>{var i=n.getAttribute("id");t.push([i,e+n.getBoundingClientRect().y])})}(),l(),d()},300)};window.addEventListener("resize",function(e){s&&clearTimeout(s),u()}),u()})();</script><script>
    var _hmt = _hmt || [];
    (function() {
      var hm = document.createElement("script");
      hm.src = "https://hm.baidu.com/hm.js?ed38f6d2bda5881f1e32b079fec99636";
      var s = document.getElementsByTagName("script")[0];
      s.parentNode.insertBefore(hm, s);
    })();
</script>

<script type="text/javascript">
    document.body.oncontextmenu = 
    document.body.ondragstart = 
    document.body.onselectstart = 
    document.body.onbeforecopy = 
    function () { return false; };
    document.body.onselect = 
    document.body.oncopy = 
    document.body.onmouseup = 
    function () { document.selection.empty(); };
</script>

</html>
